\chapter{Ramifications of  the \dnmtchipheadline methylome}
\label{chap:d:methylation}
\minitoc

During the doctoral project, the \dnmtchip mouse model\cite{Gaudet2003} was used to elaborate on the role of DNA methylation in the context of \mllafnine acute myelogenous leukemia. We sought to explore, why proper \proteinnamemouse{Dnmt1} expression is essential for self-renewal of \mllafnine leukemia stem cells in a cell-autonomous manner\cite{Vockentanz2011}. 

Two key observations characterized the \dnmtchip methylome:

\begin{enumerate}
	\item Large hypomethylated regions in lamina-associated domains\dissref{chap:d:methylation:compromisedregions}.
	\item Stable methylation at CpG-Islands\dissref{chap:d:methylation:persistentregions:cgi}.
\end{enumerate}

The observed degree of hypomethylation suggested noteworthy deleterious effects on cellular homeostasis and transcriptional regulation in particular and we wanted to elucidate the most relevant mechanisms. This chapter of the discussion shall therefore address our findings in the context of general knowledge about DNA methylation and provide possible interpretations. 

\section{Assumptions regarding the \dnmtchipheadline methylome}
\label{chap:d:methylation:shaping}

It is well established that an interplay of active and passive processes shapes a methylome. Methylation may be actively deposited or removed and is either faithfully copied or passively lost during replication\citerev{Iurlaro2017}. In case of the \dnmtchip methylome, one can safely assume that the reduced \proteinnamemouse{Dnmt1} levels will to some extent impair copying of methylation marks to the newly synthesized strand during replication and thus increase the passive methylation loss. The experimental and analytical plan of the project was conceived  based on two assumptions: 

\begin{enumerate}
	\item Our main working hypothesis was the inverse relationship between methylation persistency and the cells' division rate. We proposed that methylation loss would intensify in fast-cycling cells such as the leukemic bulk compared to slowly dividing cells such as HSCs or LSCs. In accordance with previous work\cite{Aran2011}, we conjectured that compromised regions would be prone to methylation loss as a consequence of constrained time for methylation maintenance during replication. 
	\item Numerous reports had indicated, that cancer cells can repress tumor suppressor genes by promoter hypermethylation\citerev{Rodriguez-Paredes2011}. We assumed that the fast-cycling \dnmtchip leukemic cells would be unable to maintain (or reestablish) all relevant repressive promoter methylation. Thus, tumor suppressor genes would inadvertently become reactivated when the methylation capacity of \dnmtchip was exhausted.  Therefore, we aimed to pinpoint epigenetically regulated genes, which were crucial for cancer self-renewal by intersecting the reactivated genes in all replicates. 
\end{enumerate}

Both assumptions were refuted by our own results as well was published studies over the course of the project:

\subsection{Inverse relationship of division rate and methylation persistency}
\label{chap:d:methylation:shaping:inverse}

Although there was, at large, an association of late-replication with profound methylation loss, not all compromised regions were late replicating\dissrefpage{chap:r:persistency:nextcgi}. The most important challenge, however, was that the separation into persistent and compromised regions in \dnmtchip did not arise after leukemic transformation, but was already effectuated on the level of slowly propagating HSCs\dissrefpage{chap:r:wgbs_chip_hsc:leukemia}. Neither did we observe an intensification of the methylation loss from MPP1 to MPP3 stage in healthy \dnmtchip hematopoiesis\dissrefpage{chap:r:wgbs_chip_hsc:mpp}, which would be expected if the proliferation rate would predominantly determine the methylation loss.

Along this line, a recent publication elucidated new details about the methylation copying process, such as the inheritance of most methylation marks within 20 minutes of replication\cite{Xu2018b}. For the first time, genome-wide quantitative data became available and the authors also developed a new computational method called \emph{in silico Strand Annealing}(iSA), which permitted to assemble the original DNA double strands from book-ended reads. By applying iSA, the authors characterized surprisingly large fractions of hemimethylation\footnote{CpGs-dyads containing just one methylated cytosine}, which accounted for \SIrange{4}{18}{\percent} of the DNA methylome. Remarkably, their analysis challenged the prevailing view that hemimethylation is purely transient, since they found hemimethylated sites that are stably inherited over several cell divisions dependent on \proteinname{Dnmt3b}. These sites serve as directional binding site for \proteinname{CTCF} and enable orientation-specific co-localization of methyl-binding proteins\cite{Xu2018b}.

While these results introduced another crucial regulatory function of methylation, which could hypothetically be affected in \dnmtchip, the study also demonstrated that de novo methylation can ably compensate for critical passive loss, if required. For repetitive sequences, this had been suggested before\cite{Liang2002}.

None the less, \dnmtchip methylomes clearly exhibited pronounced compromised regions, although the propagation rate of \mllafnine cells did not seem high enough to enforce critical methylation loss in the light of the typically quick restoration within minutes after replication. Taking into account that genome-wide just \SI{20}{\percent} of methylation serves a regulatory purpose\cite{Ziller2013}, it appeared likely that methylation in those compromised regions was predominantly dispensable for the \mllafnine cells. 

The discovery that cells can actively mark genomic regions where 5-methylcytosine is dispensable with \histwoarg for renouncement of methylation maintenance\cite{Veland2017} substantiated this model. Strikingly, the responsible enzyme \genename{Prmt6} was significantly upregulated in \dnmtchip \mllafnine leukemia in RNA-seq. Therefore, a future \histwoarg ChIP-seq in \dnmtchip is suggested to integrate the mark with the location and strength of the compromised regions. 

\subsection{Inadvertent reactivation of epigenetically repressed genes}
\label{chap:d:methylation:shaping:react}

When the previous PhD student, Lena Vockentanz, handed the project over,  the global mRNA sequencing of hypomethylated and control leukemia had not yet been evaluated. Therefore, it was unresolved, which gene programs promoting leukemic self-renewal might be perturbed in \dnmtchip. 

Additionally, she suggested to focus on new surface markers present on hypomethylated stem cells, since such a discovery would be of interest from a therapeutic point of view. Distinct surface markers appearing upon hypomethylation would be a valuable target for anti-cancer therapy to combine demethylating drugs with specific antibodies\cite{Vockentanz2011}.

Accordingly, we proceeded with the project, analyzed the RNA-seq and determined significantly differentially expressed genes\dissrefpage{chap:r:degenes:genotypecontrast:genes}. We also identified divergent transcriptional programs and pathways, which clearly discriminated \dnmtwt and \dnmtchip leukemia\dissrefpage{chap:r:degenes:genotypecontrast:pathways}. Our in-depth characterization also involved a \hisfourthree buffer domain analysis\dissrefpage{chap:r:degenes:bufferdomains} and elaborated on a possible elongation bias\dissrefpage{chap:r:transcription:elongation} as well as perturbed splicing\dissrefpage{chap:r:tinats:stringtie}.

Although we could detect notable differences between the genotypes, none of them could be straightforwardly linked to hypomethylated promoters or derepressed genes. Since methylation at CpG-Islands was predominantly persistent in \dnmtchip\dissref{chap:d:methylation:persistentregions:cgi}, few promoter CpG-Islands hypomethylated. Differential pathways typically comprised none or just a few downstream genes with altered promoter methylation. On a global level, reactivation of repressed genes was almost completely absent\dissrefpage{chap:r:transcription:promhypooverall} and \dnmtchip exhibited dramatically fewer active promoters than the \dnmtwt\dissrefpage{chap:r:tinats:denovolocalization}. 

While several, also recent studies, stressed reactivation of silenced promoters as possible mechanism of \genenamemouse{Dnmt} inhibitors\cite{Cai2017,Brocks2017} or \genenamemouse{Dnmt1} reduction\cite{Trowbridge2012,He2017}, it was never discriminated between active and passive demethylation. The importance of this discrimination is discussed later\dissref{chap:d:methylation:persistentregions:cgi:transmeth}.

Furthermore, the new study from Chenhuan Xu and Victor Corces\cite{Xu2018b} conclusively showed that hemimethylation was virtually absent around transcription start sites in a variety of different mouse embryonic stages. While the degree of hemimethylation in the gene body varied, it was consistently depleted at the TSS, which suggested stringent regulation. Considering that the methylation status of single CpGs within an island is tightly spatially correlated\cite{Hebestreit2013}, little room was left for gradual, passive demethylation at promoter CGIs. 

Taken together, we now favor the interpretation that the few transcripts, which exhibited hypomethylation at the promoter and concomitant transcriptional upregulation, were likely upregulated on purpose by active demethylation. This view is also backed by published literature\cite{DeSmet2010,He2019}. Deleterious methylation loss in \dnmtchip rather focused on cis-regulatory elements than promoters, which will be discussed later\dissrefpage{chap:d:enhancers:mechanism:dnmtchipgeno}.  On top of this complexity, \genenamemouse{Dnmt1} is involved in a variety of methylation-independent functions, which might have contributed to the phenotype\dissref{chap:d:strain:dnmtonealtfunct}.

\section{Characteristics of the large-scale compromised regions}
\label{chap:d:methylation:compromisedregions}

Certainly, the most distinct features of the \dnmtchip methylome were the large compromised regions. Based on our assumptions \dissref{chap:d:methylation:shaping}, we proposed that the hypomethylated regions would explain the self-renewal deficit observed in \dnmtchip \mllafnine leukemia. However, a derepression of epigenetically silenced genes was not detectable and Irina Savelyeva, a former postdoc of our laboratory, also ruled out deficits in genomic stability\dissrefpage{chap:i:abridged:project:previous_results}. 

On the other hand, she noticed evidence for senescence in \dnmtchip. Because a paper around the same time linked senescence with a methylome harboring compromised regions\cite{Cruickshanks2013}, we were intrigued to explore a potential causal relationship.  

To do so, we conducted an in-depth investigation regarding the properties of those regions. We ascertained that the compromised regions differed from the persistent areas mostly by the number of unchanged CpGs and not by the degree of demethylation. Furthermore, we made every effort to exactly localize the regions and quantify the degree of demethylation by fitting a custom generalized additive model\dissrefpage{chap:r:gam:methylpredict}, because a standard approach had failed to discriminate domain borders\dissrefpage{chap:r:comprom:methylseeker}. We could clearly show that the compromised regions were distinct from the methylation canyons described in HSCs\cite{Jeong2014}, which are located in intergenic open chromatin and related to \histhirtysixtwo\cite{Weinberg2019}.

Subsequently, we reanalyzed several third-party WGBS datasets for which other hypomethylation features had been described. We sought to explore, if the compromised regions were mechanistically identical to the \emph{partially methylated domains} (PMDs)\cite{Lister2009} or large-scale hypomethylations in cancer\cite{Timp2014} \dissref{chap:d:methylation:cchippmds}. This was required to understand, how these regions could potentially relate to the \dnmtchip phenotype\dissref{chap:d:methylation:compromisedregions:effect}. 

\subsection{Do \dnmtchipheadline methylomes harbor PMDs?}
\label{chap:d:methylation:cchippmds} %legacy reference

In a purely descriptive sense, such a statement is correct, since the compromised regions exhibited a significant increase in partial methylation. However, we have extensively reviewed the scientific literature describing methylomes with PMDs or PMD-like features (including reanalysis of some third-party datasets) and advocate that there are at least two distinct mechanisms.   

\begin{figure}[!bht] 
	\centering
	\includegraphics[width=\textwidth]{figures/output/vectors/pmdorigin.pdf} 
	\caption{Schematic representation of the four principal methylation patterns that could resemble a partically methylated domain in a WGBS dataset. Both alleles of one cell are shown in panels \panellabel{A} to \panellabel{C}, whereas \panellabel{D} depicts DNA from several cells. For the sake of simplicity, a CpG dyad on a strand is just symbolized as one circle. White color indicates no modification and black fill represents a methylated cytosine residue on the particular strand.\panellabel{A},  \panellabel{C} and  \panellabel{D} will appear as partial methylation even at base resolution, while \panellabel{B} required smoothing to be applied (which is typically the case in WGBS analysis\cite{Hansen2012,Wreczycka2017}).}
	\label{fig:pmdorigin}
\end{figure}

Thus, it may be helpful to delineate the possible methylation patters that could give rise to a partially methylated domain:  The CpG dyad is the smallest unit that can exhibit partial methylation, which is then preferably called hemimethylation	\reffigure{fig:pmdorigin}{, panel~A}. Indeed, pulse chase bisulfite sequencing experiments suggested that pronounced hemimethylation is strongly associated with partial methylation seen in conventional WGBS\cite{Xu2018b}. However, a similar pattern would also emerge, if smoothing was applied to fully methylated and unmethylated CpGs alternating on the same DNA strand\reffigure{fig:pmdorigin}{, panel~B}. It could also be that methylation is predominantly limited to one allele\reffigure{fig:pmdorigin}{, panel~C}., a scenario which occurs for example in breast cancer\cite{Hon2012}. Lastly, the pooling of hundreds and thousands of cells for a WGBS library implies that such patterns could also arise as an averaged value of absolute, but mixed methylation states in different cells\reffigure{fig:pmdorigin}{, panel~D}. On top of this complexity, these states may change dynamically\cite{Luo2018}.  

The mechanism underlying a specific PMD is often impossible to determine, unless the experiment has been designed to allow for it: In a dataset generated from a pulse chase experiment, one can distinguish newly synthesized strands and thus detect hemimethylation\cite{Xu2018b}. Other than that, it is sometimes possible to separate the alleles based on SNPs, if primary tumor material is used\cite{Hon2012}. 

Evidently, the latter approach is not feasible when inbred mouse strains with little genomic variation are used like in our case. Eventually, we opted for compromised regions, since a lack of \proteinnamemouse{Dnmt1} strongly suggested an impaired methylation inheritance across cell divisions.Therefore, the large-scale PMD-like features in \dnmtchip probably emerged as increased hemimethylation \reffigure{fig:pmdorigin}{, panel~A}, which after subsequent cell divisions deteriorated into a heterogeneous methylation pattern \reffigure{fig:pmdorigin}{, panel~B}. 

\subsection{Impact of PMD-like compromised regions}
\label{chap:d:methylation:compromisedregions:effect}

One important question was, whether and how the compromised regions can provide an explanation for the self-renewal deficits of \dnmtchip \mllafnine \kithi cells. As discussed previously\dissref{chap:d:methylation:shaping:react}, promoter hypomethylation linked to gene dysregulation was absent in \dnmtchip leukemia\dissrefpage{chap:r:transcription:promhypooverall}. However, because of the relationship of PMDs to cancer and senescence, we suspected that there might be other effects, which shall be discussed herein.   

PMDs or PMD-like features were described in early embryonal cell types\cite{Shirane2016} as well as the placenta\cite{Schroeder2013a}, in pancreatic cells\cite{Schultz2015}, in senescent cells\cite{Cruickshanks2013}, in long-term cultured cells\cite{Ziller2013} and particularly in cancer cells\cite{Hansen2011,Berman2012,Timp2014,Vidal2017}.  

\paragraph{In cancer:} In tumors, successive loss of DNA methylation during tumorigenesis seems to be the norm: From healthy tissues to the primary tumors and their associated metastases, a clear trend of hypomethylation in lamina-associated, late-replicating regions is observable\cite{Vidal2017}. The authors further conclude based on the loss of association between methylation levels of neighboring CpG sites that hypomethylation in PMDs occurs randomly rather than at distinct consecutive CpG sites\cite{Vidal2017}. Therefore, most cancer-related PMDs probably arise from pooling of heterogeneously methylated chromosomal sections in WGBS\reffigure{fig:pmdorigin}{, panels~B+D}. 

Intriguingly, presence and intensity of the PMDs does not reflect the expression level of methyltransferases. In a comprehensive study, human post-mortem samples of 18 tissue types from four individuals were investigated and no systematic expression difference of \proteinnamemouse{Dnmt1}, \proteinnamemouse{Dnmt3a}, \proteinnamemouse{Dnmt3b} and \proteinnamemouse{Dnmt3l} between samples with and without PMDs were found\cite{Schultz2015}.  

However, this observation does not preclude temporary enzyme insufficiencies in fast cycling cancer cells. Intriguingly, it has been shown that the arginine methyltransferase \genenamemouse{Prmt6}, which mediates \histwoarg deposition, is upregulated in many cancers and its depletion or inhibition restores DNA methylation in hypomethylated breast cancer cells\cite{Veland2017}. Since \genenamemouse{Prmt6} was also significantly upregulated in \dnmtchip \mllafnine leukemia, it was tempting to speculate that compromised areas might be characterized by \histwoarg deposition as a means to prioritize methylation maintenance.

Prioritizing might be relevant to \dnmtchip, since a lack of \genenamemouse{Dnmt1} could facilitate stochastic epigenetic silencing by laying down repressive histone marks at sites of fork stalling\cite{Jasencakova2010a,Nikolov2016}. This hypothesis provides a rationale, how hypomethylation could be linked to the formation of long-range repressive chromatin\cite{Hon2012}, a process which appears to play role in several cancers. The respective repressive chromatin domains were termed LOCKs\cite{Timp2013} or LRES\cite{Clark2007}. 

In this context, it should be noted that LRES exhibit hypermethylation of consecutive CGIs\cite{Clark2007}, which is reminiscent of the methylation pattern of the largest compromised regions in \dnmtchip \supple. In that sense, the compromised regions in \dnmtchip may even emerge in different ways: Regions of \SI{150}{\kilo b} rather by hemimethylation\reffigure{fig:pmdorigin}{, A $\rightarrow$ B+D}, while the larger areas preferably located near the distal ends of chromosomes might be linked to LOCKs respectively LRES\reffigure{fig:pmdorigin}{, C+D}.  

\paragraph{In senescence:} Repressive chromatin domains akin to the LOCKs/LRES in cancer were also described in senescent cells, where they are known as senescence-associated heterochromatic foci (SAHF)\cite{Narita2003}. Analogous to the process triggered by a lack of \proteinnamemouse{Dnmt1} at the replication forks\dissrefpage{chap:d:strain:dnmtonealtfunct}, oncogenes may induce DNA replication stress and trigger an \proteinnamemouse{ATR} (ataxia telangiectasia and Rad3-related)-mediated senescence response involving SAHF formation\cite{Bartkova2006}. 

Like the LOCKs/LRES domains in cancer, SAHF coincidence with large regions of partial methylation\cite{Cruickshanks2013}. This interwoven nature of repressive heterochromatin and DNA hypomethylation\reffigure{fig:pmdorigin}{, panel~C} was already noted in a remarkable forward-looking review by Bruce H. Howard published in 1996:

\begin{displayquote}
	\enquote{\emph{Interestingly, the above results fit very well with a model of cellular senescence in which an interdependence exists between DNA methylation and maintenance of heterochromatin domains.
			[...] Errors in maintenance [...] are postulated to accumulate during the proliferative life span, ultimately triggering a cell cycle checkpoint and consequent irreversible cell cycle exit. Such a heterochromatin-linked model of senescence is directly coupled to DNA replication, because maintenance of heterochromatin-like structures requires that these structures be reformed in conjunction with each traverse of the cell cycle. [...] A semistochastic character also follows simply by assuming that recession of heterochromatin-like domains is progressive and widespread, but that not all domains, when lost, trigger a cell cycle checkpoint with equal efficiency.}}\cite{Howard1996}
\end{displayquote}

Although this proposal was made years before the era of genome-wide sequencing, it was substantiated two decades later: In single cells, chromosomal compartmentalization may be abrogated\cite{Kind2015}, and lamina-associated heterochromatic domains may dissociate from the nuclear lamina in a possibly incidental manner\cite{Chandra2015}. For aging hematopoietic stem cells it was shown that the dissociation and other effects are related to the altered expression of \proteinnamemouse{LaminA/C}\cite{Espada2008,Grigoryan2018}. Accordingly, Lamin~B1 binding is redistributed in senescent cells\cite{Sadaie2013} and its depletion provokes fundamental chromatin reorganization that consolidates cell-cycle exit\cite{Shah2013,Chandra2015}.

The exact influence of DNA methylation on this detachment remains elusive up to date, but it was shown that proper nuclear organization during terminal differentiation is dependent on methyl-binding proteins\cite{Brero2005,Linhoff2015} connecting the chromatin fiber to the nuclear lamina\cite{Guarda2009}. Moreover, a recent comprehensive study profiled the methylomes of 39 diverse primary tumors and analyzed them alongside 343 additional human and 206 mouse WGBS datasets\cite{Zhou2018}. By studying PMDs in cancer, they derived a local CpG sequence context associated with preferential hypomethylation and thereby noted a previously undetected methylation loss in almost all healthy tissue types. The degree of hypomethylation reflected the cell division history and suggested that senescence is the regular endpoint of normal differentiation\cite{Chandra2015}. In contrast, other authors emphasize that terminal differentiation and senescence are two distinct processes\cite{Gorgoulis2019},  since only the latter is associated with a secretory phenotype\cite{Ito2018}. None the less, methylation in (some?) PMDs apparently serves as a mitotic clock in healthy cells, which is implicated in terminal differentiation/senescence\cite{Zhou2018}.

Intriguingly, altered expression of \proteinname{Dnmt1} can figuratively change the clock to a different time in a methylation-dependent manner\cite{Lin2014}. However, this mitotic clock barrier is generally overcome entirely during tumorigenesis for example by inactivation of the kinase ATM (ataxia telangiectasia mutated) or loss of p53\cite{Collado2005}. Accordingly, remethylation of the PMDs is not required for senescence bypass instigated by the SV40-T antigen\cite{Cruickshanks2013}. Thus, the increased heterochromatin induction observed in premalignant cells is typically retained in the tumors\citerev{DiMicco2011}.

In summary, most methylation loss occurring in the compromised regions of \dnmtchip has probably no impact on gene expression or any other gene oriented regulatory function. If the proposed mitotic clock function is correct, cells in the \dnmtchip mice could age faster and exhibit an altered formation of repressive chromatin domains as well as divergent lamina association harboring an inherent senescence risk. Eventually, the cells would become senescent sooner, unless \mllafnine transformation or contributory random mutations undermine the clocking mechanism completely. 

Ultimately, it remained elusive, if the on average  \SI{2.8}{\percent}$\,/  \,$\SI{9.3}{\percent} senescent cells (in leukemic bulk and LSC respectively), which we observed by \ensuremath{\beta}-galactosidase staining, were sufficient to justify \dnmtchip leukemia phenotype in its entirety. It should be noted, however, that this method does not suffice for the proper detection of senescent cells\cite{Gorgoulis2019}. 
 
\section{Persistent methylation at CpG-Islands and promoters}
\label{chap:d:methylation:persistentregions:cgi}

The second noteworthy features of the \dnmtchip methylome were the remarkably persistent CpG-Islands (CGIs). Because we sought to characterize the epigenetic mechanisms causing the self-renewal impairment of \dnmtchip leukemia, we mostly focused on compromised sections of the genome. 

Even though no evident hypomethylation at CpG-Islands could be determined in WGBS, they would be an interesting subject for further studies due to their considerable regulatory functions. However, single-cell methylome analysis would be required to substantiate incidences of stochastic aberrant methylation at CpG-Islands, because subclones with a truly deleterious epigenetic aberration would be quickly marginalized due to their competitive disadvantage\cite{Welch2012,Jan2013,Assenov2018}.

\subsection{Persistently unmethylated CpG-Islands} 
\label{chap:d:methylation:persistentregions:cgi:umeth}

In both leukemic methylomes, unmethylated CpG-Islands were confined to the open chromatin / interLAD areas\dissrefpage{chap:r:wgbs:lad_demethylation_cgi}. At first glance, one might be tempted to dismiss unmethylated CGIs as irrelevant to the \dnmtchip phenotype, since further passive demethylation is impossible. 

However, it may be helpful to keep in mind that an unmethylated CGI is a peculiarity. Eukaryotic genomes are typically dominated by AT\cite{Chargaff1951}, which can be explained by the spontaneous hydrolytic desamination of unmethylated cytosine to uracil. Most CpGs in genomes are methylated to better preserve them\cite{Ziller2013}, which is the actively enforced default\cite{Domcke2015}. Therefore, a CpG-Island, which inadvertently hypomethylated in \dnmtchip would not just happen to remain unmethylated ever since. 

So the unmethylated rather than the methylated state of a CpG-Island demands explanation and active regulation. Recent studies have shown, that DNA secondary structures are heavily implicated in maintaining regulatory CpG-Islands in an unmethylated state: G-quadruplex (G4) structures tightly bind and sequester \proteinnamemouse{Dnmt1} away from certain CGIs and prevent their methylation\cite{Mao2018}.  However, it would be inaccurate to consider G-quadruplex structures solely as decoys for \proteinnamemouse{Dnmt1}, since they exert a wealth of different regulatory activity\citerev{Tian2018,Mukherjee2019}. The expression of \proteinnamemouse{c-MYC}\cite{Panda2015} as well as \kit\cite{Phan2007} is for example regulated by G-quadruplex structures at the respective promoters. Another secondary structure diverting from the regular double-helix strand is the i-motif, which can be formed in cytosine-rich DNA and might also serve regulatory purposes in vivo\cite{Zeraati2018}.

Although true structural complexity of DNA in vivo is just being revealed, unmethylated CpG-Islands are clearly hot spots of such uncommon formations. Particularly in the light of the so far underappreciated precise temporal orchestration of DNA methylation, which was lately uncovered by single-cell techniques\citerev{Luo2018}, it is therefore recommended to revisit the DNA methylation of CpG-Islands in \dnmtchip \mllafnine leukemia with single-cell techniques. 

\subsection{Methylation transition at CpG-Islands} 
\label{chap:d:methylation:persistentregions:cgi:transmeth}

Transitions of the methylation state of a CGI are tightly controlled. For promoter CpG-Islands, it has been shown that \proteinnamemouse{EZH2} and \proteinnamemouse{PRC2/3} recruit DNA methyltransferases to cease the expression of genes\cite{Vire2006}. Lineage commitment and differentiation are for example typically accompanied by a successive gain in methylation\cite{Luu2013,Rulands2018}. During commitment, the cells achieve the permanent silencing of genes mediating stemness or governing alternative lineage development by promoter methylation and by the placement of repressive histone marks. This process has been proven for ES-cells\cite{Xie2013,Lee2014} as well as hematopoiesis\cite{Cabezas-Wallscheid2014}. 

On the other hand, differentiation also requires the activation of previously silenced genes. Active demethylation is mostly attributed to oxidative reversal\citerev{Shen2014a} by the TET proteins\citerev{Pastor2013}.  However,  also \proteinnamemouse{Dnmt3a} and \proteinnamemouse{Dnmt3b} are capable of active demethylation\cite{Chen2012}\citerev{Wijst2015}.  

Remarkably, TET proteins play an important role in the regulation of hematopoietic malignancies\citerev{Han2015} and MLL-rearranged leukemia. Through coordination with MLL-fusion proteins, \proteinnamehuman{Tet1} acts and reactivates critical co-targets such as \proteinnamemouse{Hoxa9},\proteinnamemouse{Meis1} and \proteinnamemouse{Pbx3}\cite{Huang2013,Huang2016a}. Rarely, leukemia cases are reported that even exhibit a direct fusion of \proteinnamehuman{Tet1} to \proteinnamemouse{Mll1}\cite{Lorsbach2003,Lee2013}. 

It should be stressed, that active demethylation fosters the deposition of epigenetic marks, which are important for the integrity of the \hisfourthree depositing \proteinnamehuman{Set1/COMPASS} complex. TET proteins associate with the O-GlcNAc transferase (EC 2.4.1.255, OGT), which glycosylates histone~2B (\proteinnamehuman{H2B}), host cell factor~1 (\proteinnamehuman{HCF1}) and other proteins\cite{Chen2013,Deplus2013}. Since these glycosylations promote the deposition of histone~2 K120 monoubiquitination\cite{Fujiki2011} and ultimately \hisfourthree, it is ensured that transcription is initiated as a result of the active demethylation of a promoter. 

A lack of those additional epigenetic cues was probably responsible for the absence of functional gene reactivation after passive demethylation in \dnmtchip\dissref{chap:d:methylation:shaping:react}. This interpretation is strongly backed by a study in fibroblasts, where passive promoter demethylation after shRNA-mediated knockdown of \proteinnamemouse{Dnmt1} intensified in areas of  lower chromatin accessibility and did seldom translate into direct expression changes\cite{He2019}. Instead, a lion's share of the reactivation could by be clearly attributed to active DNA demethylation by the Ten-eleven translocation methylcytosine dioxygenase~1(\proteinnamehuman{Tet1})\cite{He2019}.

\subsection{Persistently methylated CpG-Islands} 
\label{chap:d:methylation:persistentregions:cgi:meth}

Methylated CpG-Islands could be observed in the open as well as lamina-associated domains of the \mllafnine genome.  At promoters, methylated CpG-Islands are clearly associated with transcriptional repression\citerev{Bouras2019}. However, CGIs are not limited to the promoter of genes, but may also occur in introns or be found within the coding sequence itself.  Methylation of such CGIs reduces physical interaction with promoters, abates bivalent chromatin and results in transcriptional activation of key regulatory genes such as \proteinnamemouse{PAXs}, \proteinnamemouse{HOXs} and \proteinnamemouse{WNTs}\cite{Lee2017}. Therefore, the persistent methylation of CpG-Islands in \dnmtchip could not only serve as repressive mark but also support the expression of other genes. 

To identify CpG-Islands, which specifically need to be preserved in a methylated state, ubiquitinylation of H3 at lysines K18 and K23 could be monitored in \dnmtchip. If \histwoarg is absent, but \hisninethree present, \genenamemouse{Uhrf1} establishes \hiseighteenub and \histwentythreeub, which promote DNA methylation inheritance by \genenamemouse{Dnmt1}\cite{Harrison2016,DaRosa2018, Li2018a}. Further histone marks are likely also implicated \citerev{Rose2014}. It is conceivable that such marks will allow to pinpoint critical regulatory CpG-Islands in \dnmtchip.   

\section{Methylation-independent roles of Dnmt1}
\label{chap:d:strain:dnmtonealtfunct}

Lastly, it should be pointed out that there are methylation-independent functions of \proteinnamemouse{Dnmt1}. These need to be considered, when DNA hypomethylation is induced by hypomorphic \proteinnamemouse{Dnmt1} mouse strains as well as the established \proteinnamemouse{Dnmt1} inhibitors. 

Studies investigating the cellular transcriptome after treatment with DNA methyltransferase inhibitors (DNMTis) have repeatedly reported effects unrelated to direct promoter DNA hypomethylation\cite{Schmelz2005,Flotho2009}. DNMTis are typically cytosine nucleoside analogues, which become integrated into the DNA and form stable adducts with \proteinnamemouse{Dnmt1}\cite{Christman2002}. The enzyme becomes irreversibly bound to 5-Aza-2'-deoxycytidine (Decitabine) residues in the DNA, which eventually confers cytotoxicity. Therefore, the therapeutic effect of DNMTis in cancer therapy is possibly methylation-independent\cite{Juettermann1994}.

Normally, \proteinnamemouse{DNA (cytosine-5)-methyltransferase 1} is recruited to replication foci by interacting with  \proteinnamemouse{Pcna} and  \proteinnamemouse{Uhrf1}\cite{Easwaran2004,Esteve2006} and is loaded onto hemiCpGs to methylate the nascent cytosines during DNA replication. It is well established that \genenamemouse{Dnmt1} knockdown triggers intra-S-phase arrests\cite{Milutinovic2003} or activates stress response checkpoints such as ataxia telangiectasia mutated-Rad3-related (\proteinnamemouse{ATR})\cite{Unterberger2006}. Apparently, removal of \proteinnamemouse{Dnmt1} from replication forks is the trigger for these responses, since ectopic expression of \proteinnamemouse{Dnmt1} lacking a functional catalytic domain alleviated the stress response\cite{Unterberger2006}. This notion is also backed by the observation that senescence in IMR90 fibroblasts can be overcome by the SV40 T-antigen, while the characteristic hypomethylated methylome is retained\cite{Cruickshanks2013}.  

Although we observed on average \SI{2.8}{\percent} senescent cells in the hypomorphic leukemic bulk (\SI{9.3}{\percent} in LSCs), it remained unclear to what extent such a stress response has relevance for the \dnmtchip strain. The negative \ensuremath{\gamma}H2AX-stainings performed by Irina Savelyeva challenged an \proteinnamemouse{ATR}-mediated response and most \dnmtchip cells were arrested in G1-phase\dns, whereas a lack of \proteinnamemouse{Dnmt1} typically causes an intra-S-phase cell cycle arrest\cite{Milutinovic2003}. Both observations seemed to argue against a pronounced stress response in \dnmtchip.

However, acute replication stress challenges proper chromatin restoration even below the threshold that results in a cell cycle arrest\citerev{Nikolov2016}. It facilitates stochastic epigenetic silencing by laying down repressive histone marks at sites of fork stalling\cite{Jasencakova2010a}. Unfortunately, we did not assay repressive chromatin in \dnmtchip, but the striking absence of unannotated TSS clusters in corresponding areas\reffigurepage{fig:tinatdensplot_LSChicpc1overall.pdf}{, bottom row} might reflect increased compaction of chromatin and diminished cellular plasticity. 

Additionally, the persistently methylated CpG Islands in compromised, mostly heterochromatic regions\dissrefpage{chap:r:persistency:nextcgi} could be indicative of long-range epigenetic silencing (LRES) in the hypomorphic mice, a process with particular relevance for carcinogenesis\cite{Clark2007}. Because of the excessive heterchromatin formation, \dnmtchip \mllafnine leukemic stem cells would, according to a model by the Feinberg lab\cite{Pujadas2012}, respond poorly when challenged by variable conditions. Therefore, a lack of \proteinnamemouse{Dnmt1} might cause improper chromatin restoration after cell divisions resulting in a survival and self-renewal bias in \dnmtchip independently of the methylation levels.

Apart from a possible heterochromatin-spreading in \dnmtchip, the manifold interactions of \proteinnamemouse{Dnmt1} with histone-modifying enzymes demands attention\citerev{Qin2011}.

These predominantly repressive chromatin modifiers allow \proteinnamemouse{Dnmt1} to alter transcription of target genes independent of DNA methylation in a direct manner\cite{Espada2011,Clements2012}.  Given that many upregulated transcripts did not feature a hypomethylated promoter\dissrefpage{chap:r:degenes:promhyposingle}, it was therefore conceivable that also methylation-independent functions of  \proteinnamemouse{Dnmt1} constituted to the phenotypic alterations of the \dnmtchip strain.    