\chapter{Transcriptional regulation in leukemia}
\label{chap:d:enhancers:mechanism}
\minitoc

The intricate balance between transcriptional control and noise is pivotal for cancerogenesis\citerev{Pujadas2012}. Undoubtedly, enhancers contribute strongly to this process and thus to cellular homeostasis in general. By identification of enhancers with relevance for leukemia, we presumed to find possible cues explaining the impairment of \dnmtchip leukemic stem cells. Since enhancers may be subject to carcinogenic methylation changes\cite{Rasmussen2015,Aran2013,Aran2014}, the \dnmtchip genotype might complicate establishment and maintenance of such pathogenic alterations\citerev{Herz2014,Smith2014}. 

For this reason, we investigated the presumably active enhancers in \mllafnine leukemia\dissrefpage{chap:r:enhancers:calling} and derived important commonalities \dissrefpage{chap:r:enhancers:motifs}. Herein, we will discuss the most relevant aspects of this endeavor and implications for leukemogenesis in a \dnmtchip background. 

\section{Establishment of our \mllafnine enhancer catalog}
\label{chap:d:enhancers:mechanism:catalog}

A few years ago, it was not yet well known that enhancers contribute to leukemogenesis\dissrefpage{chap:i:abridged:enhancers:leukemia} and that malignant transformation often involves perturbation of enhancer activity\citerev{Sur2016}. By now, hematopoietic enhancers have been characterized at single-cell and single-variant resolution\cite{Corces2016,Ulirsch2019}, but the corresponding datasets were not yet available when we conducted our project. 

Back in 2014, the laboratory of Ido Amit had just published the first comprehensive enhancer study in healthy hematopoiesis\cite{Lara-Astiaso2014}, but similar data for leukemia was still missing. Therefore, we compiled our own catalog for \mllafnine leukemia\dissrefpage{chap:r:enhancers:calling} and identified \num{6386} and \num{6662} putative enhancers in \dnmtwt and \dnmtchip respectively. Surprisingly, the majority of them (\SI{82.45}{\percent}) was specific for either of the genotypes\reffigurepage{fig:enhancers:enhancercanidates_venn_manual}{}.
 
The sheer abundance of distinct sites suggested a relevant fraction of false positive enhancers and called for extra caution in handling the data. Therefore, we hierarchically clustered the sites to derive repeatedly occurring characteristic signatures\dissrefpage{chap:r:enhancers:cluster}. We presumed that a common chromatin signature across healthy hematopoietic cell types\dissrefpage{chap:r:enhancers:cluster:clades:healthy} mirrored a concordant regulation and indicated close functional ties.  

Particularly, such subclusters (referred to as \emph{clades}) caught our attention, which consisted of significantly more CAGE-defined enhancers than was expectable by chance\dissrefpage{chap:r:enhancers:cluster:clades:clearydata}. Such clades exhibited a considerable enrichment of \hisfourthree, \hiseighteenac and \histwentysevenac as well as RNA.Pol.II in \mllafnine leukemia\reffigurepage{fig:enhancers:cleary_lymphpoidprog_clades}{}	\reffigurepage{fig:enhancers:cleary_lymphpoidprog_oddsratio}{} and were frequently confirmed by ATAC-seq 	\reffigurepage{fig:enhancers:trowbridge_atac_percent_representation_abs}{}. In particular, the \hisfourthree mark was quite unusual and shall be discussed below in greater detail\dissref{chap:d:enhancers:mechanism:properties:hisfourthree}. 

As a downside of this strategy, we mostly disregarded putative enhancers in the \amitten cluster\dissrefpage{chap:r:enhancers:cluster:kmeans}. It comprised CAGE-defined enhancers that were mostly devoid of chromatin modifications in healthy hematopoiesis. Thus, we considered them as a mixture of mostly false positive calls with a minority of leukemia-specific enhancers, which we could not address adequately.  

\section{Notable characteristics of our enhancer catalog}
\label{chap:d:enhancers:mechanism:properties}

\subsection{Rarity of leukemia-specific enhancers}
\label{chap:d:enhancers:mechanism:properties:rarity}

As stated in the previous section, the high disagreement between the putative enhancer sets of the two genotypes prompted us to incorporate data from the healthy hematopoiesis to reduce the number of false positive sites under consideration. In return, we deliberately accepted the consequences of missing out on some leukemia-specific enhancers.

The rationale behind this approach was that leukemia-specific enhancers in human leukemia typically arise from somatic mutations. For instance, such mutations may introduce a novel super enhancer upstream of the TAL1 oncogene and sustain its expression\cite{Mansour2014,Vahedi2015}. 

Thus, it normally takes a preleukemic hematopoietic stem cell, which has progressively acquired an increasing mutation burden over its lifetime, to identify a relevant number of aberrant enhancers\cite{Corces2016}. 

Littermates of a mouse model, on the other hand, are by design as similar as possible on the genetic level. Accordingly, the search for somatic mutations is unpromising, but has none the less been attempted for \mllafnine leukemia\cite{George2016}. On the other hand, some leukemic clones are propagated several times in recipient mice or kept for longer periods in vitro and therefore might exhibit significant somatic mutations. Additionally, long-term culture is associated with pronounced genomic demethylation\cite{Ziller2013}, which could reactivate dormant enhancers\cite{Aran2013,Aran2014}.        

Despite the limited applicability of a mouse model to investigate leukemia-specific enhancers, we noted a relevant set of \hisseventyninetwo positive sites within the tenth cluster. This chromatin modification is deposited by \proteinnamehuman{Dot1l} and is required to protect elements from a repressive protein complex composed of \proteinnamemouse{Sirt1} and \proteinnamemouse{Suv39h1} in leukemia\cite{Chen2015}. Since binding of the fusion proteins \mllaffour or \mllafnine recruits \proteinnamehuman{Dot1l} with great efficiency\cite{Kuntimaddi2015}, strong \hisseventyninetwo marks typically suggest oncoprotein binding\cite{Prange2017}. Such \proteinnamehuman{MLL}-targeted enhancers have been termed  KEEs\cite{Godfrey2019} and preservation of their open chromatin state is pivotal to uphold the leukemic differentiation block\cite{Cusan2018}.  

However, reanalysis of published data\cite{Bernt2011} showed that neither \hisseventyninetwo nor direct binding by \mllafnine \dissrefpage{chap:r:enhancers:motifs:mlltwo} were associated with or enriched in any particular cluster or clade. Furthermore, the normalized \hisseventyninetwo signal tended to be stronger in \kitlow than in \kithi cells\reffigurepage{fig:enhancers:cleary_cluster_enrichment}{, bottom row}, which implied that KEEs might be less relevant for the leukemic stem cells (LSC) than for the bulk leukemia.

KEEs also eluded a closer examination, since the chromatin interaction data used to assign the targeted promoters originated from the HPC-7 murine blood stem/progenitor cell model\cite{Wilson2016}. Therefore, leukemia-specific putative enhancers were typically not assigned to target transcripts\dissrefpage{chap:r:enhancers:targets:assignment}, which is why hardly any putative enhancers from the \amitten cluster were recorded in the top interactions despite their large number \supple. 

In summary, the rarity of leukemia-specific enhancers in our catalog was attributable to usage of a mouse model instead of patient material as well as technical limitations such as the lack of chromatin interaction data from leukemia.  

\subsection{\hisfourthree as hallmark of particular enhancer clades}
\label{chap:d:enhancers:mechanism:properties:hisfourthree}

The classical histone mark signature linked to active enhancers is the combination of \hisfourone and \histwentysevenac\cite{Heintzman2007,Ernst2011,CaloWysocka2013}. Therefore, it is the most used pattern to screen ChIP-seq datasets for the presence of enhancers and was used by the laboratory of Ido Amit to cluster their hematopoietic enhancer catalog\cite{Lara-Astiaso2014}. Consequently, we modeled our clustering strategy on the same data and approach\dissrefpage{chap:r:enhancers:cluster}.

Hence, we were surprised to find \hisfourthree to be characteristic of a variety of highly accumulated clades of the clusters \amitthree, \amitfive, \amiteight and \amitnine in \mllafnine leukemia \dissrefpage{chap:r:enhancers:cluster:clades:clearydata}. By absolute numbers, the majority of \hisfourthree-positive enhancers were assigned to clades of cluster \amitnum{3} \reffigurepage{fig:enhancers:cleary_lymphpoidprog_clades}{, top row}, whereas enhancers from insignificant clades were consistently low in \hisfourthree \reffigurepage{fig:enhancers:cleary_cluster_enrichment}{, top tow}. We were inclined to elaborate on this finding, since \hisfourthree plays an important role in leukemogenesis\dissref{chap:d:enhancers:mechanism:hisfourthreeleukemiaimportance}. 

While it was irritating in the first place, the presence of \hisfourthree and \hisfourone was not contradicting as every nucleosome consists of two histones~H3 and may thus carry both marks simultaneously. Yet, \hisfourthree is commonly believed to be restricted to gene promoters instead of enhancers. This belief is put in question by unified models seeking to overcome the distinction between promoters and enhancers\cite{Andersson2015a,Andersson2015}. Furthermore, \hisfourthree has been regarded as the best indicator of active enhancers in lymphoid lineages\cite{KochAndrau2011,Koch2011}, while \hisfourone is not required for correct enhancer function in Drosophila\cite{Rickels2017}. 

Taken together, both \hisfourone as well as \hisfourthree mark active enhancers, but \hisfourone is relatively universal while \hisfourthree only characterizes a subset of enhancers. About a third of those \hisfourthree enhancers is also bound by \proteinnamehuman{CTCF} as illustrated by a comprehensive study of murine chromatin states\reffigure{fig:mouse_chromatin_states}{}\cite{Bogu2015}.

Aforementioned study distinguished two major enhancer classes mostly based on the occurrence of \hisfourthree. Conversely, those enhancers might be targeted by different histone-lysine N-methyltransferases. While Drosophila has just three K4 methyltransferases\footnote{ \genenamemouse{dSet1}, Trithorax (\genenamemouse{Trx}) and Trithorax-related (\genenamemouse{Trr})}, mammals possess six COMPASS-like complexes: \genenamemouse{Setd1a}  and \genenamemouse{Setd1b}, Mll1 (\genenamemouse{Kmt2a}) and Mll2 (\genenamemouse{Kmt2b}) as well as Mll3 (\genenamemouse{Kmt2c})  and Mll4 (\genenamemouse{Kmt2d})\citerev{Shilatifard2012,Piunti2016}. 

\begin{figure}[t] 
	\centering
	\includegraphics[width=0.7\textwidth]{figures/output/vectors/boguchromhmm-figa.pdf} 
	\caption{Using both RNA-seq and ChIP-seq data from eight murine tissues (brain, heart, liver, kidney, spleen, small intestine, testes and thymus) as well as mouse embryonic stem cells, a comprehensive chromatin state map was computed by the group of Marc Marti-Renom\cite{Bogu2015}. Reprinted here is\panellabel{Figure~3A} of this publication. Each cell in the table denotes the percentage of cases in which a given ChIP-seq peak is found at genomic positions corresponding to a specific chromatin state.}
	\label{fig:mouse_chromatin_states}.
\end{figure}

At regular gene promoters the trimethylation is typically established by the \proteinnamehuman{Set1/COMPASS} complex, of which two variants with either \genenamemouse{Setd1a} or \genenamemouse{Setd1b} exist\cite{Lee2007}. The non-overlapping nuclear localization of the two variants suggests that both exert non-redundant functions\cite{Lee2007}.

Targeting of the \proteinnamehuman{Set1/COMPASS} complex is mainly mediated by \genenamemouse{Wdr82}, which recognizes  the Ser5-phosphorylated C-terminal domain of RNA polymerase II\cite{Lee2008} and thereby directs it to sites of active transcription in a histone \proteinnamehuman{H2B} ubiquitination-dependent manner\cite{Wu2008a}.  Intriguingly, \genenamemouse{Wdr82} has also been shown to be responsible for the active termination of enhancer RNAs (eRNAs).  Upon \genenamemouse{Wdr82} depletion, enhancers abundantly spawned long and non-coding RNAs due to termination defects\cite{Austenaa2015}.

Because we identified our enhancers based on active bidirectional eRNA transcription\dissrefpage{chap:r:enhancers:cage}, a \genenamemouse{Wdr82}-mediated recruitment of a histone-lysine N-methyl\-trans\-ferase complex appeared to be likely. Since \genenamemouse{Wdr82} is missing in the \proteinnamehuman{Mll/COMPASS} complexes\cite{Wu2008a} and is limited to the \proteinnamehuman{Set1/COMPASS} complexes, the latter seemed to be responsible for the \hisfourthree marks at the respective enhancers. However, CXXC-type zinc finger protein~1 (\proteinnamemouse{Cfp1}), which is also part of \proteinnamehuman{Set1/COMPASS}\cite{Lee2005,Clouaire2012,Cao2016} did not bind to the motif in ChIP-seq \dns.

None the less, the \hisfourthree deposition was indeed mediated by a CXXC domain, albeit that of  \genenamemouse{Kmt2b} (\proteinnamemouse{Mll2}\dissrefpage{chap:r:enhancers:motifs:mlltwo}. Hence, the CG-rich motifs \dissrefpage{chap:r:enhancers:motifs:tfs:accu} recruited \proteinnamehuman{Mll2/COMPASS} to deposit \hisfourthree at those enhancers. 

\section{\hisfourthree, Mll2, CXXC-domains and leukemia}
\label{chap:d:enhancers:mechanism:hisfourthreeleukemiaimportance}

\paragraph{\hisfourthree:} Around the same time when we first observed the abnormal \hisfourthree pattern at the accumulated enhancers, the group of Micheal Cleary reported that the oncogenic potential of \mllafnine LSCs was mainly regulated by high-level \hisfourthree marks at promoters\cite{Wong2015}. In contrast, \hisseventyninetwo was low in LSCs, but increased upon differentiation into \kitlow blast cells. At this step, also the KEEs come into play\dissref{chap:d:enhancers:mechanism:properties:rarity}. 

Intriguingly, \hisfourthree is also implicated in the leukemic potential of \genenamemouse{Nup98} fusions, in which an extrinsic plant homeodomain (PHD) finger targets the joint protein to \hisfourthree marked regions of the genome\cite{Wang2009}. When mutations in the PHD fingers abrogate \hisfourthree binding, the leukemic transforming capability is lost, since the differentiation-associated polycomb-mediated removal of the mark can no longer be prevented\cite{Wang2009}. Later, it was also shown that \genenamemouse{Nup98} not only blocks removal of \hisfourthree, but can also recruit the Wdr82-Set1A/COMPASS complex to mediate deposition of \hisfourthree \cite{Franks2017}.  Subsequently, \hisfourthree triggers the \histwentysevenac modification\cite{Morgan2017a}. 

\paragraph{Mll2:} Hence, the mechanism for deposition of the mark was of great interest to us. \proteinnamehuman{Mll2/COMPASS} could not be considered likely, as it is dispensable for self-renewal in embryonic stem cells\cite{Glaser2006}. However, its deletion impairs the differentiation of ES cells into primordial germ cells (PGCs)\cite{Lubitz2007}, the precursors for the oocytes and spermatozoa. Seemingly unrelated at the first glance, leukemia and germ cells in reality are closely connected\citerev{Ratajczak2017}. Germ cells may give rise to leukemia\cite{Ladanyi1990} and AML and CML cells express sex hormone receptors and respond to stimulation with gonadotrophins\cite{Abdelbaset-Ismail2016}. 

Therefore, we deemed the ChIP-seq data from ES cells / PCGs\cite{Hu2017} applicable to infer binding of \proteinnamemouse{Mll2} at the accumulated enhancers in \mllafnine \dissrefpage{chap:r:enhancers:motifs:mlltwo}.  In oocytes, \proteinnamehuman{Mll2/COMPASS}  primarily targets distal cis-regulatory elements for \hisfourthree deposition\cite{Hanna2018}, but may also aim at some bivalent promoters in PGCs\cite{Hu2017}. 

In \mllafnine leukemia, \proteinnamemouse{Mll2} is indeed pivotal for self-renewal and maintenance of the leukemic stem cell\cite{Chen2017a}. It is expressed at least as abundantly as  \proteinnamemouse{Mll1}in AML, including both, MLL-rearranged and other subtypes\cite{Chen2017a}. The knock-out of  \proteinnamemouse{Mll2}  is deleterious to the leukemia in vivo\cite{Chen2017a} by promoter as well as enhancer-mediated effects\dissrefpage{chap:r:enhancers:targets:mlltwotargets}.

As illustrated by a different response to menin inhibition\cite{Chen2019}, \proteinnamemouse{Mll2} does not jointly act with \mllafnine at the same sites. Furthermore, an artificial \proteinnamemouse{Mll2} fusion protein is unable to transform hematopoietic cells in vitro\cite{Bach2009}. Despite structural conservation, no other \proteinnamemouse{Mll} homolog can replace \proteinnamemouse{Mll1} in such leukemogenic fusion proteins, which is attributed to differences in their CXXC-domain. 

\paragraph{CXXC-domain:}  This domain generally binds to unmethylated CpG-dinucleotides and is found in a variety of chromatin-associated proteins\cite{Xu2018a} with different chromatin binding properties and functions\citerev{Long2013}. The CXXC-domain is retained in all known \proteinnamemouse{Mll1} fusion proteins (\mllfp)\citerev{Slany2016}, but lacked entirely by \proteinnamemouse{Mll3} and \proteinnamemouse{Mll4}\cite{Xu2018a}. Subtle differences between the CXXC-domain of \proteinnamemouse{Mll1} and \proteinnamemouse{Mll2} preclude an oncogenic potential of the latter in the context of fusion proteins\cite{Bach2009}.   

In vitro, gel shift experiments indicate almost indistinguishable DNA-binding properties\cite{Bach2009}, but in vivo a divergent nuclear localization and function of \proteinnamemouse{Mll1} and \proteinnamemouse{Mll2}  is evident\cite{Bach2009}. Experimentally, mutagenesis of the CXXC-domain of \proteinnamemouse{Mll2} to mimic the binding properties of \proteinnamemouse{Mll1} has been attempted\cite{Birch2013}. Yet, domain-swapping with the CXXC-domains of other proteins has shown that solely the CXXC-domain of Dnmt1 is functionally equivalent to that of  \proteinnamemouse{Mll1} and elicits leukemia in the context of a \mllfp\cite{Risner2013}. 

Remarkably, a \mllfp is not necessary to confer leukemogenic capacity, if the CXXC-domain of \proteinnamemouse{Mll1} is disfigured. In roughly \SIrange{5}{10}{\percent} of AML or ALL cases,  \proteinnamemouse{Mll1} is mutated by chromosomal translocations with other proteins, while partial tandem duplication (PTD) of \proteinnamemouse{Mll1} constitutes another \SIrange{5}{10}{\percent}\cite{Somervaille2010}.  In PDT leukemia, either a part of the CXXC-domain or the full region is duplicated\citerev{Basecke2006} and the HOX gene cluster becomes activated due to intensified binding\cite{Dorrance2006}. Because of the two CXXC-domains, this type of leukemia resembles a lot the subgroup of \proteinnamemouse{Mll1} fusions with other CXXC-proteis such as \proteinnamemouse{LCX}\cite{Ono2002}. PTD leukemia has an extremely poor outcome\cite{Choi2018}, but critically depends on contributory mutations for leukemogenesis\cite{Yip2017}.

Early additional mutations in PTD leukemia commonly affect the methylome of cells. For example \proteinnamemouse{Dnmt3a} is frequently affected\cite{Kao2015}, but also \proteinnamemouse{IDH1}, \proteinnamemouse{IDH2} and \proteinnamemouse{TET2}\cite{Sun2017}. Mutations in the latter three typically cause a genomic hypermethylation phenotype\citerev{Im2014}. Since the CXXC-domain is methylation-sensitive, this emphasizes the importance of abnormal binding to CG-motifs in the pathogenesis of such leukemia. 

\section{Implications for the \dnmtchipheadline genotype}
\label{chap:d:enhancers:mechanism:dnmtchipgeno}

Most aspects of the \dnmtchip genotype have been discussed previously\dissrefpage{chap:d:methylation}. However, two topics with respect to enhancers are still missing. Firstly, how methylation may affect enhancer activity itself; and secondly, how demethylation may perturb enhancer-promoter pairs. 

\subsection{Methylation determines enhancer activity}
\label{chap:d:enhancers:mechanism:dnmtchipgeno:methenh}   

In hematopoiesis, enhancers prime a cell far before the actual lineage separation takes place during differentiation\cite{Paul2015,Drissen2016,Buenrostro2018}. These findings are in accordance with the notion that enhancer signatures characterize cell types superiorly and foreshadow future gene expression programs\cite{Lara-Astiaso2014,Arner2015,Corces2016}. 

Also cancer predisposition is to a good extent reflected by pre-neoblastic alterations at the enhancer sites. These can be either genetic such as mutations\cite{Pomerantz2009,Wasserman2010,Mansour2014,Bahr2018} and translocations\cite{Yamazaki2014,Groeschel2014}) or epigenetic. Because active enhancers are typically unmethylated or lowly methylated\cite{Stadler2011}, the prevalent pre-malignant or malignant epigenetic alteration is hypermethylation\cite{Aran2013,Aran2014,Rasmussen2015,Planello2016}).  In a way, hypermethylation represents a restoration of the default methylated state, whereby de novo methyltransferases outcompete methylation-sensitive transcription factors such as \proteinnamehuman{NRF1}\cite{Domcke2015} and decommission the enhancer.  

However, it would be oversimplified to assume that a methylated enhancer can not contribute to the regulation of gene expression, since there are also many transcription factors that preferably bind to methylated sites\cite{Yin2017a}. Furthermore, unmethylated degenerate binding sites sequester the transcription factor \proteinnamehuman{EGR1} away to non-functional locations\cite{Kemme2017}.

The aforementioned mechanism is of great interest with regard to the \dnmtchip genotype, since Irina Savelyeva noticed an intriguing enrichment of \proteinnamehuman{EGR1} motifs within the promoters of genes that were downregulated in \dnmtchip \mllafnine leukemia. Because of the rather random demethylation,  it is much easier to rationalize such a decoy-based mechanism in \dnmtchip than a spurious reactivation of a specific decommissioned enhancer. The latter requires a precise loss of methylation at the enhancer site, whereas degenerate \proteinnamehuman{EGR1} motifs are quite common in the genome such that an arbitrary loss of  \proteinnamehuman{MeCP2} binding would suffice for an impairment. 
However, we could not explain the \proteinnamehuman{EGR1} motif enrichment in the data back then and mainly focused on upregulated genes.   

Further results also suggested that methylation levels of the motif \motifmlltwo were subject to specific regulation, possibly to govern \proteinnamehuman{Mll2/COMPASS} binding\dissrefpage{chap:r:enhancers:motifs:methylation}. It was tempting to speculate that also for this quite frequent and degenerate motif, a sequestration takes place, since we observed a hypomethylation of the motif and a significant redistribution of the \hisfourthree mark in \dnmtchip \dissrefpage{chap:r:degenes:bufferdomains}. 

\subsection{Methylation affects chromatin organization}  
\label{chap:d:enhancers:mechanism:dnmtchipgeno:methloops}  

By and large, enhancers either act through RNAs\citerev{Chen2017d} or by increasing the concentration of transcriptional activators near a gene promoter. This mechanism of enhancer action typically requires a change of the three-dimensional chromatin structure and the formation of a DNA loop - a process that is still incompletely understood. The discovery of preexisting chromatin looping, which precedes the actual signaling\cite{Jin2013} has challenged the previous model\cite{Arensbergen2014} that solely specific transcription factors govern the looping\cite{Drissen2004,Vakoc2005}. Importantly, the size of chromatin loops may be altered without dissociation of the cohesin complex\cite{Haarhuis2017}.

In any case, it is evident that the formation and dissociation of loops must undergo tight sequential coordination, since a gene is typically targeted by several cis-regulatory elements and one element may also be involved in the regulation of different genes\cite{Hughes2014,Bertolino2016,Javierre2016}. This is further illustrated by chromatin loop collisions in cells lacking the cohesin-unloading factor \proteinnamehuman{WAPL}\cite{Allahyar2018}.

Since \proteinnamehuman{CTCF}-mediated chromatin interactions are influenced by stably inherited hemi-methylation that flanks \proteinnamehuman{CTCF} motifs\cite{Xu2018b}, a perturbation of the latter in \dnmtchip seems possible. In gliomas with IDH mutations and a hypermethylation phenotype, insulator dysfunction due to abrogated \proteinnamehuman{CTCF}-binding allows for a enhancer-driven increased expression of the receptor tyrosine kinase \proteinnamehuman{PDGFRA}, a prominent glioma oncogene\cite{Flavahan2016}. Lacking actual Hi-C data from \mllafnine leukemia or the \dnmtchip genotype, we could not verify if enhancer-promoter pairs were truly malformed in \dnmtchip. 

In general, \proteinnamehuman{CTCF}  is not considered to be relevant for chromatin compartmentalization, which is rather attributed to cohesin\cite{Haarhuis2017}. Specifically for the HOX gene cluster, however, it has been shown that \proteinnamehuman{CTCF} subdivides it into euchromatin and facultative heterochromatin\cite{Narendra2015} and that some \proteinnamehuman{CTCF}-binding sites serve as promoters for functional lncRNAs, which impact chromosomal interactions\cite{Nwigwe2015}. Considering the tremendous importance of the HOX gene cluster, particularly \genenamemouse{HOXA9}, for leukemia\cite{Fujino2001,Adamaki2015,Mohr2017}, a misregulation in \dnmtchip might have deleterious effects. The GAM-derived methylation persistency suggested a breakdown of the chromatin compartmentalization at some positions\reffigurepage{fig:wgbs_gam_diff_chr14}{, black arrow}, but this needs to be verified by additional data\footnote{such as Lamin- DamID\cite{Vogel2007}}.

\section{Outlook}
\label{chap:d:methylation:outlook}

Currently, the continuation of the project is not scheduled. Upon advancement, the readjusted scope of the project would determine the subsequent steps and experiments.

\paragraph{Role of methylation and its relationship to senescence:} The PMD-like compromised regions in \dnmtchip might affect the chromatin association with the nuclear lamina and interfere with the mitotic clock \dissref{chap:d:methylation:compromisedregions:effect}. Eventually, chromatin in aging hematopoietic stem cells dissociates from the nuclear lamina and the higher-order chromatin architecture collapses\cite{Grigoryan2018}. The effect of the altered lamina composition has already been shown\cite{Grigoryan2018}, none the less the \dnmtchip strain could represent an interesting model system to study a prematurely aging hematopoiesis.

To better assess the scientific potential of this project, comprehensive WGBS datasets of mouse embryonic fibroblasts (MEFs) could be reanalyzed and integrated. By now, methylome data of MEFs after \proteinnamemouse{Dnmt1} knockdown (GSE93058) and throughout the regular cell cycle (GSE92903) have been published\cite{He2017,He2019}. Therefore, after integration with previously published lamina-association data from MEFs\cite{Meuleman2013}, it would be possible to elaborate on the variability and fluctuation of methylation marks in the context of LADs and interLADs. 

Ultimately, proper chromatin maps and matched methylome data of aged \dnmtchip HSCs would be required for an authoritative study. Considering that the Vaquerizas group in Münster possesses comprehensive skills to elucidate the chromatin architecture from challenging cell types and little source material\cite{Hug2017}, a local collaboration would be possible. \clearpage   

\paragraph{Methylation in \mllafnine leukemia and novel therapeutic targets:} The initial focus of the project was the identification of novel, potentially druggable targets for the treatment of AML. We proposed that hypomethylation at promoters in \dnmtchip would lead to a reactivation of tumor suppressor genes, which we would be able to single out and target specifically for therapeutic purposes. However, it turned out that inadvertent reactivation of genes was implausible considering the absence of hemimethylation at gene promoters and lack of supportive epigenetic marks required for transcription\dissref{chap:d:methylation:persistentregions:cgi}. The \dnmtchip mouse strain is therefore probably an unsuitable model system for this approach. 

On top of that, methylation-independent effects due to a stress response are to be expected in \dnmtchip leukemia. Hence, the ectopic expression of \proteinnamemouse{Dnmt1} lacking a functional catalytic domain would be recommended to alleviate a potential stress response\dissrefpage{chap:d:strain:dnmtonealtfunct}. The same applies to the classical DNA methylation inhibitors 5-Azacytidine and 5-aza-2'-deoxycytidine, which are by no means more suitable, since they trap \proteinnamemouse{Dnmt1} irreversibly at the DNA\cite{Juettermann1994} and also trigger a stress response\cite{Christman2002}. Therefore, either approach harbors the risk to wrongly attribute effects to DNA demethylation that are actually related to other functions of \proteinnamemouse{Dnmt1}\cite{Juettermann1994}.     

To study the effect of demethylation in \mllafnine specifically, another approach is hence needed. Targeting the adenosylhomocysteinase \genenamemouse{Ahcy}\cite{Turner1998,Fumic2007} instead of \proteinnamemouse{Dnmt1} might be such an alternative. It is the only enzyme capable of hydrolyzing S-adeno\-syl-L-homo\-cysteine\footnote{\raggedright See Gene Ontology category GO:0004013. In spite of its similarity with \genenamemouse{Ahcy}, recombinant S-adenosylhomocysteine hydrolase-like protein~1 \genenamemouse{Ahcyl1} ectopically expressed in bacteria neither affects the enzyme activity of \genenamemouse{Ahcy} nor does it itself exhibit hydrolase activity\cite{Ando2003}.}, which is generated during the DNA methylation process\cite{Yang2003}. Since S-adenosyl-L-homocysteine is a strong inhibitor of \proteinnamemouse{Dnmt1}, the knockdown or inhibition of \proteinnamemouse{Ahcy} in \mllafnine leukemia should impair DNA methylation without triggering a stress response, since the replication fork complex can still be faithfully assembled.   

However, manipulation of \genenamemouse{Ahcy} may also have unwanted side effects.  A study suggested that the downregulation of adenosylhomocysteinase might actually promote tumorigenesis\cite{Leal2008}, while strong overexpression can trigger apoptosis due to unphysiologic accumulation of adenosine\cite{Hermes2008}. Besides DNA methylation, it is also implicated in mRNA cap methylation\cite{Fernandez-Sanchez2009}. The latter mechanism may predominantly underlie the efficacy of \genenamemouse{Ahcy} inhibitors, which are in preclinical development for the treatment of \proteinnamemouse{c-Myc}-driven tumors\cite{Uchiyama2017}.

Taken together,  knockdown or inhibition of \genenamemouse{Ahcy} may be a different approach to investigate hypomethylation on a genome-wide scale with fewer side-effects. In contrast,  CRISPR-dCas9 based epigenome-editing tools could be used to add or remove methyl-residues in a site-specific manner\citerev{Pulecio2017}, if the effects of hypo- or hypermethylation at specific promoters or cis-regulatory element are of interest. 