\chapter{Introduction}
\minitoc

\section{\mllafnine and MLL-rearranged leukemia}
\label{chap:i:abridged:leukemia}

The \mllafnine  fusion protein is a recurrent leukemogenic genetic abnormality, which arises from the chromosomal translocation t(9;11)(p21.3;q23.3). This results in a gene fusion of the \genenamehuman{Kmt2a} gene (\proteinnamehuman{Mll1} protein) with the \genenamehuman{Mllt3} gene (\proteinnamehuman{Af9} protein)\cite{Ziemin-vanderPoel1991,Thirman1993}. 

\mllafnine fusions are part of a larger group, the \genenamehuman{Kmt2a}-rearranged leukemia\citerev{Li2014a}. To date, more than 60 fusion partners~[\citenum{VanderBurg1999}, reviewed in \citenum{Slany2016}] and multiple breakpoints\cite{Kobayashi1993} have been reported for the \genenamehuman{Kmt2a} gene / \proteinnamehuman{Mll1} protein, which is one of several vertebrate homologues of the \emphspecies{Drosophila} positional identity regulator \proteinnamedrosophila{Trithorax}\cite{Tkachuk1992,Djabali1992} and thus implicated in H3K4 methylation. 

Although the first \genenamehuman{Kmt2a} fusion was reported as additional recombination in chronic myelogenous leukemia (CML)\cite{Rowley1973}, they are much more prevalent in pediatric acute myelogenous leukemia (AML)\cite{Sorensen1994,Balgobind2009}. \genenamehuman{Kmt2a} rearrangements in general are found in roughly 40\% of the infant \footnote{\numrange{0}{3} years of age} cases of acute myeloid leukemia, but constitute to less then 5\% of the cases within the in the AYA group\footnote{adolescents and young adults, \numrange{15}{39} years old} and are even more scarce in older patients\cite{Bolouri2017}. In elderly patients, \genenamehuman{Kmt2a} rearrangements are very rare and occur almost exclusively during relapses as side effect of leukemia treatment with topoisomerase inhibitors\cite{Super1993}. 

The \mllafnine rearrangement specifically can be found in 9.5\% of childhood and in 0.5\% of adult acute myeloid leukemia\cite{Balgobind2009}. Therefore, it is recognized as a separate entity by the current WHO classification of myeloid neoplasms and acute leukemia and is considered as one of the eleven subcategories within \emph{AML with recurrent genetic abnormalities}\footnote{AML with t(9;11)(p21.3;q23.3);MLLT3-KMT2A}\cite{Arber2016}.

Because the \proteinnamehuman{AF9} protein, which is encoded by the \genenamehuman{Mllt3} gene, is a component of the super elongation complex (SEC)\cite{He2010,Lin2010}, \mllafnine leukemia is mostly characterized by a dysregulation of transcriptional elongation\citerev{Mohan2010}. The downregulation of \genenamehuman{Mllt3} in cultured  haematopoietic stem-cells impairs self-renewal and negatively affects engraftment efficiency\cite{Calvanese2019}.
 
Human \proteinnamehuman{MLL-AF9} can also transform mouse cells and educe a well characterized leukemia model system\cite{Somervaille2009,Krivtsov2013,Stavropoulou2016} out of several early hematopoietic lineages\cite{George2016}. Importantly, it is known that the expression of \proteinnamehuman{CD117}/\kit is expedient to enrich for leukemia stem cells (LSCs)\cite{Dick2005} from the leukemic bone marrow\cite{Krivtsov2006,Somervaille2006}. Such LSC-enriched \kitpos fractions of independently established MLL-AF9 leukemia were investigated in the present study. 

\section{DNA methylation in acute myeloid leukemia}
\label{chap:i:abridged:amlmethylation}

Since 1925, the occurrence of methylated cytosine\footnote{also known as 5-methylcytosine (5-mC) and epi-cytosine} in nucleic acid extracts of bacteria is known\cite{Johnson1925}, while its incorporation in regular nucleotides of eukaryotes could be shown in 1951\cite{Cohn1951}. Because of the high frequency of 5-methylcytosine (5-mC) in genomes, the term DNA methylation typically refers to this base, although other bases such as adenine can be modified accordingly (N6-methyladenine )\cite{Dominissini2012}\citerev{Ji2018}.    

In the subsequent decades, 5-mC DNA methylation was shown to be involved in various, mostly repressive functions such as X chromosome inactivation, imprinting and the silencing of endogenous retroviruses as well as regular genes\citerev{Schuebeler2015}. Despite its role in long-term silencing, localized DNA methylation is more dynamic than originally believed and can be deposited or removed by various enzymes in a timely manner\citerev{Luo2018}. In humans and mice, the methylation of 5-mC in genomic DNA is performed by \proteinnamemouse{Dnmt1}, \proteinnamemouse{Dnmt3a} and \proteinnamemouse{Dnmt3b}\citerev{Lyko2018}, whereas the TET enzymes\citerev{Pastor2013} mediate the oxidative reversal\citerev{Shen2014a}. Whether these further oxidized bases 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC) and 5-carboxycytosine (5caC) are just intermediates in the reversal process or have distinct regulatory functions is still disputed\citerev{Plongthongkum2014}. Remarkably, also \proteinnamemouse{Dnmt3a} and \proteinnamemouse{Dnmt3b} may be implicated in the active demethylation of DNA\citerev{Wijst2015}.   

The measurement of DNA methylation is typically performed by high-throughput bisulfite sequencing\cite{Meissner2005}, which can  nowadays be scaled down to single cells\cite{Mulqueen2018}. Bisulfite treatment will cause a conversion of unmethylated cytosine bases to uracil, which are replaced by thymine during a subsequent PCR amplification. Those bases will appear as mutations with regard to a reference genome and hence the methylation rate can be calculated from the ratio of mutated vs. reference reads after the alignment\citerev{Wreczycka2017}. Alternatively, enrichment-based methods like MeDIP-seq and MRE-seq are available to measure DNA methylation\cite{Xing2018}. 

In 1983, it was shown that aberrant DNA hypermethylation may cause thalassaemia\cite{Kioussis1983}, which was the first pathogenic DNA methylation to be described. Subsequently, an association of altered DNA methylation with several diseases was shown\citerev{Bergman2013}. In cancer, the silencing of tumor-suppressor genes by DNA hypermethylation at promoters is widespread\citerev{Rodriguez-Paredes2011}, although DNA hypomethylation seems to be the norm on a genome-wide scale\cite{Vidal2017}. Aberrant DNA methylation is also implicated in the pathogenesis of acute myeloid leukemia (AML), in which epigenetic peculiarities exert a strong influence\cite{Eriksson2015} due to fewer mutations than most other cancers\cite{TCGAC2013}.  Yet, mutations in genes related to DNA methylation are quite common\cite{TCGAC2013} and various AML subtypes display distinct methylation profiles\cite{Figueroa2010,Akalin2012a}. For example, haploinsufficiency of \proteinnamemouse{Dnmt3a} enhances self-renewal\cite{Challen2014} of hematopoietic stem cells and predisposes them to myeloid malignancies\cite{Cole2017}, possibly by hypomethylation of the intergenic euchromatin space\cite{Weinberg2019}. Contrarily, pathogenic hypermethylation is observed in AML with mutations in genes like \proteinnamehuman{IDH1}/$\,$\proteinnamehuman{IDH2}\cite{Im2014}, \proteinnamehuman{BCAT1}\cite{Raffel2017} or \proteinnamehuman{WT1}\cite{Rampal2014} or the \proteinnamehuman{Tet} enzymes\cite{Lorsbach2003,Moran-Crusio2011,Solary2014}. Mechanistically,  hypermethylation infers with the binding of transcription factors such as \proteinnamehuman{PU.1}\cite{Sonnet2014} or \tfcebpa\citerev{Schoofs2013a}, which results in a differentiation block. 

\section{Enhancers}
\label{chap:i:abridged:enhancers:enhancerdef}

\subsection{Enhancers in general}
\label{chap:i:abridged:enhancers:general}

Operationally, enhancers augment the activity of a nearby promoter and are enriched in recognition motifs for sequence-specific transcription factors. Since the orientation and to a large extent also the exact position relative to the promoter is insignificant, they are considered to be orientation- and position-independent. Anyhow, most enhancers are thought to reside in the vicinity of the targeted promoter, although long-range contacts are possible\cite{Sanyal2012,Mifsud2015} as illustrated by the \proteinnamehuman{myc} locus \cite{Fulco2016} or the \proteinnamehuman{shh} gene\cite{Lettice2003}. One gene is typically targeted by several cis-regulatory elements and one enhancer may also be involved in the regulation of different genes\cite{Hughes2014,Bertolino2016,Javierre2016}. 

The mechanism of enhancer action requires a change of the three-dimensional chromatin structure and the formation of a DNA loop\cite{Arensbergen2014}. Initially, it was believed that specific transcription factors initiate the loops directly between enhancers and promoters\cite{Drissen2004,Vakoc2005}, but more recent models favor preexisting loops that dynamically slide long the chromatin fibre until a specific contact is established\cite{Jin2013,Haarhuis2017}.

Once enhancer and promoter have converged, a variable cascade of events is triggered that ensures transcriptional initiation or pause release. The most important player in this cascade is the mediator complex\cite{Kornberg2005,Robinson2016}, but many other coactivators\cite{Kaiser1996} are implicated as well, while regulatory cues are provided by epigenetic marks on histones\cite{Ogryzko1996,Spencer1997}. Ultimately, phosphorylations in the c-terminal tail of \proteinname{RNA polymerase II} control the transcriptional activity\cite{Hirose2007}. 

Bringing enhancer-bound protein factors close to the promoter-bound preinitiation complex (PIC) is the best characterized function of enhancers, but not the sole. Active enhancers may give rise to bidirectional transcripts termed enhancer RNAs (eRNA)\cite{Kim2010,DeSanta2010,Zhu2013a}. eRNAs were soon shown to be capped on the 5\lq end, short (\textless\SI{1}{kb}), bidirectional, unspliced and rapidly degraded by the exosome\cite{Koch2011,Andersson2014,Core2014}, which contested a relevant functional role of the pervasive transcription initiating from enhancers. 

Contrarily, eRNAs have been demonstrated to stabilize enhancer promoter association at steroid hormone response genes\cite{Wang2011,Li2013a}, to be of importance for \hisfourone and \hisfourtwo deposition by \proteinnamehuman{Mll3} and \proteinnamehuman{Mll4} at \emph{de novo} enhancers\cite{Kaikkonen2013} and to be subject to functional methylation\cite{Aguilo2016}. Thus, they exert meaningful roles at least for a subset of enhancers \citerev{Lam2014} and their expression generally correlates with their target genes\cite{Arner2015}.

Intriguingly, the majority of lncRNAs originate from enhancer-like elements~[\citenum{Su2014a}, reviewed in \citenum{Chen2017d}]. Previously the lack of splice donors\cite{Fong2001} proximal to enhancers was believed to preclude productive elongation of eRNAs\cite{Core2014}, but it was shown that they are mostly actively terminated\cite{Austenaa2015}. The main reason seems to be the prevention of convergent transcription\cite{CQuaresma2016,Flynn2016}, which triggers strong DNA-damage signaling\cite{Meng2014}. Particularly at super enhancers, which harbor clustered enhancer elements, such \proteinname{RNA polymerase II} collisions would inevitably occur upon elongation of eRNAs and thus their timely termination is pivotal\cite{Meng2014}. 

The propensity of an enhancer to generate eRNA transcripts as well the signature of histone marks at the site\cite{Ernst2010,CaloWysocka2013} depends on the state: closed, primed, poised or active \citerev{Heinz2015}. Since some of the next-generation sequencing based methods rely on these patterns to identify enhancers genome-wide, sensitivity and specificity of the respective method will vary and sometimes confine itself to enhancers in a particular state\citerev{Lim2018}. 

\herestoyoufrank{
 \item Classes of cis-regulatory elements.
 \item Mode of action, enhancer states and activation.
 \item Discovery and function of enhancer RNAs (eRNAs).
 \item Methods for genome-wide identification of enhancers.
 \item Principles of pathogenic enhancer aberrations.
}

\subsection{Enhancers contribute to leukemogenesis}
\label{chap:i:abridged:enhancers:leukemia}

Hematopoiesis, the development of diverse mature blood cells from hematopoietic stem cells requires an intricate regulation. The appropriate expression of key transcription factors such as \proteinnamehuman{PU.1}, \proteinnamehuman{GATA1}, \proteinnamehuman{GATA2} or \tfcebpa at various stages governs progenitor commitment and differentiation. Ten-thousands of enhancers are presumably involved in hematopoietic regulation in total\cite{Lara-Astiaso2014,Ulirsch2019,Bresnick2019}.

Generalizations about leukemogenesis are almost futile, given the many different subtypes. \genenamehuman{MYC} and its enhancers, however, are recurrently implicated in various leukemia as well as other cancers\cite{Zuber2011,Shi2013,Bahr2018}. In contrast, the downregulation of \proteinnamehuman{PU.1} is restricted to hematopoietic cancerogenesis. None the less, it represents a proven route to leukemia\cite{Rosenbauer2004,Metcalf2006} and already subtle \proteinnamehuman{PU.1} reduction by a heterozygous deletion of an enhancer was sufficient to initiate a myeloid-biased preleukemic state\cite{Will2015}.

Especially late-onset leukemia are characterized by the presence of preleukemic hemato\-poietic stem cells, which have progressively acquired an increasing mutation burden over their lifetime. These cells are not yet leukemic and expansive, but exhibit spurious alterations in their gene expression programs and enhancers, which increase susceptance to uncontrolled cellular expansion\cite{Corces2016}. 

Unsurprisingly, preleukemic states are heavily promoted by aberrant super-enhancers, since they govern the activation of whole gene clusters. The introduction of binding motifs for the \proteinnamehuman{MYB} transcription factor by somatic mutations forms a novel super enhancer upstream of the TAL1 oncogene and sustains its expression\cite{Mansour2014,Vahedi2015} in T cell acute lymphoblastic leukemia (T-ALL). In a particularly dismal ALL subtype driven by \proteinnamehuman{TCF3-HLF}, the chimeric transcription factor activates an enhancer cluster controlling expression of the \genenamehuman{MYC} gene and instigates the respective transcriptional program\cite{Huang2019}. Because hematopoietic \genenamehuman{MYC} expression is intricately regulated by combinatorial and additive activity of individual enhancer modules within this cluster\cite{Bahr2018}, a dysregulation of \genenamehuman{MYC} program can be mediated by various factors or arise from amplifications within the enhancer region\cite{Shi2013}. Therefore, the enhancer cluster is complicated in many leukemia subtypes and also pivotal for \mllafnine-driven AML\cite{Bahr2018}. 

A different mode of action has been reported for a distinct subtype of acute myeloid leukemia. In AML with the \emph{inv(3)(q21;q26)} karyotype\cite{Arber2016} a genomic rearrangement repositions a distal hematopoietic enhancer of \proteinnamehuman{GATA2} in close proximity to the stem-cell regulator \proteinnamehuman{EVI1}, which is ectopically activated. Concomitantly, \proteinnamehuman{GATA2} expression is diminished and both events facilitate leukemic expansion\cite{Yamazaki2014,Groeschel2014}. 

\section{Previous findings}
\label{chap:i:abridged:project:previous_results}

The Rosenbauer laboratory has a long-standing interest in the role of DNA-methylation for normal and abnormal hematopoiesis\cite{Broeske2009,Vockentanz2010}. Lena Vockentanz, a former PhD student\cite{Vockentanz2011} and Irina Savelyeva, a previous postdoc in the laboratory, conducted many experiments, which have collectively shown that \proteinnamemouse{Dnmt1} expression is essential for cell-autonomous activity of \mllafnine leukemia cells. 

Using a \polyic-inducible \mllafnine \cremx \ensuremath{\times} \dnmtfloxchip mouse model, the rate-limiting impact of diminished \proteinnamemouse{Dnmt1} levels on leukemia development was shown. However, non-excised \dnmtfloxchip cells, which had escaped induction, typically outgrew their rearranged cognates in prolonged experimental settings. Thus, this model appeared non-optimal for studying the function of leukemic stem cells (LSCs) in particular and \mllafnine leukemia with a \proteinnamemouse{Dnmt1} hypomorphic background was created using bone marrow cells from \dnmtchip mice\cite{Li1992,Tucker1996,Gaudet2003} as donors.

Consistent with the previous \mllafnine \cremx \ensuremath{\times} \dnmtfloxchip results, animals transplanted with \dnmtchip \mllafnine leukemia fell ill significantly later than those of the wild type control group\footnote{median latency \num[separate-uncertainty = true]{140.8\pm37}\,days versus \num[separate-uncertainty = true]{89.8\pm24.1}\,days after transplantation}. Animals with end-stage leukemia exhibited massive infiltration of donor \mllafninegfp cells into the bone marrow and spleens of recipient mice. Like the \dnmtwt control, \dnmtchip \mllafnine leukemia mimicked \cdelevenbpos granulocyte-macrophage progenitors and in part expressed the precursor marker \cdonehundretseventeenpos (\kit). The latter was useful to enrich LSCs from the leukemic bone marrow\cite{Krivtsov2006,Somervaille2006} and limiting dilution assays showed that the rate of LSCs in leukemia with \dnmtchip genotype was significantly lower\cite{Vockentanz2011}. 

This indicated that the prolonged latency was attributable to cell-autonomous functions of LSCs instead of microenvironment- or engraftment-deficiencies. The latter possibility was firmly ruled out by a short-term (\SI{20}{\hour}) engraftment assay, which detected that a comparable number of transplanted cells of both genotypes had infiltrated the hematopoietic organs. Furthermore the recipients for the transplantation experiments were all wild-type mice, which corroborated engraftment independence of the observations. 

Altogether the results argued for an impaired self-renewal of LSCs, which was confirmed by in vitro serial replating in methyl-cellulose. To determine, if the reduced self-renewal was functionally linked to an altered cell cycle, Hoechst\,\num{33342} incorporation was monitored: \dnmtchip \mllafnine exhibited a \SI{31}{\percent} reduction in cell numbers for LSCs in the S-G2-M phases, but no increase in apoptosis. Hence, a proportion of \dnmtchip LSCs accumulated in the non-cycling G1 phase. By quantitative PCR experiments, this G1 arrest could in part be attributed to an increased expression of the two transcripts \genenamemouse{p19/Arf} and \genenamemouse{p16/Ink4A} at the \genenamemouse{Cdkn2a} locus in \dnmtchip leukemia. Both gene products are known to accompany senescence in murine and human cells. Indeed, variable fractions of senescent cells could be detected in the \dnmtchip group by \ensuremath{\beta}-galactosidase (\ensuremath{\beta}-gal) staining. While non-leukemic hematopoietic stem/progenitor populations of \dnmtchip were devoid of senescent cells \footnote{despite known aberrant hematopoiesis and disordered lymphoid lineage development\cite{Broeske2009}}, the leukemia bulk contained up to \SI{7}{\percent} (avg. \SI{2.8}{\percent}) and the LSC fraction up to \SI{53}{\percent} (avg. \SI{9.3}{\percent}) senescent cells. Although the rate of senescence was highly variable across replicates, these findings suggested a relevant inherent senescence risk of \dnmtchip \mllafnine leukemia and provided a first plausible route to cell cycle exit and self-renewal defects. 

\section{Aim of this thesis}
\label{chap:i:abridged:project:aim}

Although the first hypothesis to explain the prolonged latency of \dnmtchip \mllafnine leukemia had been drafted, it remained elusive how the senescence program was triggered by reduced \proteinnamemouse{Dnmt1} levels in the first place. Since chemical inhibitors of DNA methylation, such as decitabine, which has received market authorization by the European Medical Agency, have proven therapeutic efficacy for the treatment of acute myeloid leukemia\cite{Stresemann2006,Hollenbach2010}, we assumed a common methylation-dependent mechanism.

The treatment with inhibitors results in an undirected reduction in DNA methylation. However, it is generally presumed that most methylation changes occur silently and therapeutic effects are only conferred, when yet to be characterized key sites have been affected by random. Our mouse model seemed to be suitable to aid the identification of those key sites, as it permitted to reduce DNA methylation by genetic \genenamemouse{Dnmt1} deficiency instead of inhibitor treatment and thus allowed to circumvent possible side-effects. We utilized the \dnmtchip mouse strain to elicit acute myeloid leukemia by transduction of \mllafnine and asked, how selective pressure and impaired methylation maintenance would shape the leukemia methylome.

The \kitpos sorted, leukemic stem cell fractions were subjected to extensive, genome-wide characterization by next-generation sequencing experiments:

\begin{itemize}
	\item Whole-Genome Bisulfite sequencing (WGBS) to assay DNA methylation
	\item RNA-seq to study gene expression changes and alternative splicing
	\item CAGE-seq to detect aberrant transcriptional initiation and call enhancers
	\item \hisfourthree ChIP-seq to corroborate active transcription and identify broad peaks, which are referred to as buffer domains and mark cell identity genes\cite{Benayoun2014}.
\end{itemize} 

The bioinformatic analysis and interpretation of the gathered data from these experiments was the centerpiece of the project. To quantify the methylation persistence across large regions and detect regional trends, a novel method for WGBS data comparison based on Generalized Additive Models was developed. To address anomalous enhancers, which have emerged as important factors in leukemogenesis\dissref{chap:i:abridged:enhancers:leukemia}, a comprehensive characterization of bivalently transcribed active enhancers and their respective methylation status was performed. 

All results were placed in context with published third-party datasets \dissrefpage{chap:ap:thirdpartydata}, which were often reanalyzed from scratch to assure full comparability with our own data. Selected genes and enhancers were also experimentally tested in vitro by shRNA knock-down or CRISPRi for their effect on self-renewal and growth rate. 