\subsection{Gene body methylation}
\label{chap:d:methylation:persistentregions:genebodies}

In contrast to promoters, which lack hemimethylation around transcription start sites, gene bodies exhibit variable degrees of hemimethylation in various developmental stages\cite{Xu2018b}. 

Since it had been shown that gene body methylation varies according to replication timing and that early-replicating genes are hypermethylated relative to late-replicating genes\cite{Aran2011}, a striking resemblance to the general \dnmtchip methylome was evident. 

Moreover, a set of nearly \num{2000} commonly hypomethylated genes that occupies highly specialized genomic, epigenomic, evolutionary and functional niches was shown to be significantly more prone to cancer-associated hypomethylation and mutation\cite{Mendizabal2017}. 

Thus, it was tempting to speculate that the faithful inheritance of gene body methylation was negatively affected in the \dnmtchip mouse model and gene bodies would gradually demethylate with cell divisions. Since widespread aberrant alternative splicing across tumors was documented\cite{Kahles2018}, we conjectured that similar effects might occur in hypomethylated leukemia. 

However, we could rule out relevant effects on the mRNA elongation efficiency\dissrefpage{chap:r:transcription:elongation} and splicing\dissrefpage{chap:r:tinats:denovoexpression}. None the less, we did not exhaustively address all possible consequences that could arise from disturbing the two major functions of gene body methylation:

\begin{mydescription}{0.5cm}
	\item [Intragenic CGIs] Genic CpG-Islands are not limited to the promoter of genes, but may also occur in the gene body. Most intragenic CpG-Islands (iCGIs) are located in introns, although they may also be found within the coding sequence itself. It has been shown that iCGIs are associated with lineage control genes and may reside in bivalent chromatin containing both active \hisfourthree and repressive \histwentyseventhree marks\cite{Lee2017}. Methylation of the iCGIs reduces physical interaction with the promoters, abates bivalent chromatin and results in transcriptional activation of key regulatory genes such as \proteinnamemouse{PAXs}, \proteinnamemouse{HOXs} and \proteinnamemouse{WNTs}\cite{Lee2017}.
	\item [Binding sites of chromatin organizers] It is essential that the DNA interaction of higher-order chromatin organizers such as \proteinnamehuman{CTCF} occurs in a directional manner, otherwise gene regulation may be perturbed. \proteinnamehuman{CTCF} , which defines TAD boundaries and shapes chromatin loops\citerev{Ong2014a}, requires stably inherited hemi-methylation rotation-symmetrically surrounding its binding motifs\cite{Xu2018b}.  Also the  methyl-CpG-binding domain (MBD) family of proteins such as \proteinnamemouse{Mecp2}, \proteinnamemouse{Mbd1a}, \proteinnamemouse{Mbd1b}, \proteinnamemouse{Mbd2a} and \proteinnamemouse{Mbd2t} show methylation-dependent\cite{Baubec2013} orientation-specific co-localization with hemi-methylation\cite{Xu2018b}. Binding occupancy diminishes with decreasing (hemi-)methylation levels, which is illustrated by the \proteinnamemouse{Mecp2} protein implicated in the pathogenesis of the Rett syndrome, a neurodevelopmental disorder\cite{Kinde2016}. Intriguingly, \proteinnamemouse{Mecp2} binding is additionally reliant on non-CpG methylation in the CA context\cite{Kinde2016}. 
\end{mydescription}  

Based on those two regulatory mechanisms, a several perturbations in \dnmtchip were conceivable: For example, it was reported that heterochromatic LRES\dissref{chap:d:methylation:compromisedregions:effect} can spread to neighboring topologically associating domains after CTCF binding is lost, causing chromatin domain boundaries to perish\cite{Forn2013}. Hence, genes in \dnmtchip might inadvertently silenced by spreading repressive chromatin marks similar to position-effect variegation\cite{Wakimoto1998}. On the contrary, deleterious demethylation could also alter enhancer-promoter interactions and reactivate silent genes. 

However, it was shown that the compartmentalization into active and inactive genome compartments remains intact upon CTCF depletion\cite{Nora2017}, challenging radical effects on the chromatin architecture in \dnmtchip resulting for example in senescence. On top of that, chances are that transcriptional dysregulation and TAD reformation rather aids than impairs the acquisition of malignant properties\citerev{Valton2016}. 

The most convincing argument against a noteworthy involvement of the gene body methylation in the \dnmtchip phenotype, however, was the extensive literature that mainly linked the DNA methyltransferases 3a and 3b to it. 

In murine embryonic stem cells (mESCs), the disruption of the PWWP domain of \proteinnamemouse{Dnmt3b} led to a decreased gene body DNA methylation at transcribed genes\cite{Baubec2015}. The same phenomenon was observed after  
ablation of the \histhirtysixthree-methyltransferase \proteinnamemouse{Setd2}\cite{Baubec2015}. Yet, the amount of DNA methylation in gene bodies does not correlate with \histhirtysixthree levels\cite{Nanan2017}. Thus, the main function of \proteinnamemouse{Dnmt3b} in gene bodies may be to maintain the hemimethylated CTCF binding sites\cite{Xu2018b}.
Importantly, the MÃ¼ller-Tidow lab demonstrated that forced overexpression of \proteinnamemouse{Dnmt3b} in \mllafnine and particularly \mycbcltwo leukemia impairs LSC function\cite{Schulze2016}.

In contrast, \proteinnamemouse{Dnmt3a} preferably colocalizes with \histhirtysixtwo at non-coding regions of euchromatin\cite{Weinberg2019}. However, if the \histhirtysixtwo methyltransferases \proteinnamemouse{Nsd1} and \proteinnamemouse{Nsd2} are deleted and is \histhirtysixtwo depleted, \proteinnamemouse{Dnmt3a} is redistributed to the \histhirtysixthree-positive gene bodies and hypermethylates them, while intergenic DNA hypomethylates\cite{Weinberg2019}.  

The importance of the carefully gauged \proteinnamemouse{Dnmt3a} balance between gene bodies and intergenic space is illustrated by its involvement in tumorigenesis. Deletion of \proteinnamemouse{Dnmt3a}  in a mouse lung cancer model accelerated tumor progression\cite{Gao2011} and genome-wide methylome analysis unveiled hypomethylated intergenic regions in euchromatin\cite{Raddatz2012}.  In acute myeloid leukemia, \proteinnamemouse{Dnmt3a} is frequently mutated\cite{Eriksson2015,Rau2016} and haploinsufficiency of the enzyme suffices to predispose hematopoietic cells to myeloid malignancies, although the exact mechanism remains elusive\cite{Cole2017}.  

In normal hematopoietic stem cells, synergistic deletion of \proteinnamemouse{Dnmt3a}/$\,$\proteinnamemouse{Dnmt3b} resulted in enhanced self-renewal\cite{Challen2014}. The underlying mechanism is unclear, but could be related to excess demethylation by one of the TET family 5-methlycytosine (5mC) hydroxylases. The group of Margaret Goodell showed that knock-down of \proteinnamemouse{Dnmt3a} in HSCs causes spreading of lowly methylated regions referred to as \emph{canyons} into intergenic regions\cite{Jeong2014}. Since canyon borders are demarked by 5-hydroxymethylcytosine, active demethylation can be assumed\cite{Jeong2014}. It is conceivable that the absence of DNA methyltransferase~3a, an imbalance between remethylation and demethylation causes inadvertent gene activation. 

While gene bodies clearly harbor DNA methylation of regulatory importance, it is mainly a matter of   \proteinnamemouse{Dnmt3a}/$\,$\proteinnamemouse{Dnmt3b} to establish, maintain, renew and shape it\citerev{Lyko2018}. Therefore, the influence of \proteinnamemouse{Dnmt1} should be limited and no major perturbations of gene body methylation in \dnmtchip are to be expected.