\paragraph{Making a case for hypermethylation:} The role of methylation for acute myeloid leukemia (AML) is ambivalent\citerev{Schoofs2013}. For example, hypomethylation caused by haploinsufficiency of \proteinnamemouse{Dnmt3a}\footnote{\proteinnamemouse{Dnmt3a} is also frequently mutated in AML\cite{Eriksson2015,Rau2016}} suffices to predispose hematopoietic cells to myeloid malignancies\cite{Cole2017} and enhances self-renewal\cite{Challen2014}. 

On the other hand, the silencing of tumor-suppressor genes by promoter hypermethylation is a hallmark of many tumors\citerev{Rodriguez-Paredes2011,Bouras2019} and also relevant for AML\cite{Sonnet2014}. This is particularly evident in the subgroup of Ten-Eleven Translocation-2 (\proteinnamehuman{Tet-2}) mutant AML.

Complete \proteinnamehuman{Tet-2} loss or haploinsufficiency increases hematopoietic stem cell self-renewal and myeloid transformation\cite{Moran-Crusio2011}. Heterozygous loss, which is commonly observed in myeloid malignancies, confers increased self-renewal to stem/progenitor cells and to extramedullary hematopoiesis in vivo\cite{Moran-Crusio2011}.  Mechanistically, \proteinnamehuman{Tet-2} insufficiency hinders the demethylation of (and \hisfourthree deposition at) key lineage determination genes such as \proteinnamehuman{PU.1} target genes\cite{Rica2013} and results in a differentiation block \citerev{Solary2014}. 

Intriguingly, genomic hypermethylation was also observed in leukemia patients having mutations in the isocitrate dehydrogenase enzymes \proteinnamehuman{IDH1}/$\,$\proteinnamehuman{IDH2}\cite{Im2014}. The mutated enzymes produce large amounts of the metabolite 2-hydroxyglutarate (2HG)\cite{Dang2009}, which is a competitive inhibitor of multiple $\alpha$-KG-dependent dioxygenases, including the TET family of 5-methlycytosine (5mC) hydroxylases and several histone demethylases\cite{Xu2011a}. Thus, leukemia patients with \proteinnamehuman{IDH1}/$\,$\proteinnamehuman{IDH2} mutations recapitulate in part the pathogenic DNA hypermethylation seen in \proteinnamehuman{Tet-2} mutated leukemia, although histone hypermethylation is also involved\cite{Kernytsky2015}. This is also corroborated by clinical trials of the inhibitors ivosidenib (targets \proteinnamehuman{IDH1}) and enasidenib (targets \proteinnamehuman{IDH2})\cite{Stone2017}.

Similarly, high levels of the BCAA transaminase~1 (\proteinnamemouse{Bcat1}) and excess activation of the branched-chain amino acid (BCAA) pathway mimics the effects of IDH mutations in AML\cite{Raffel2017}. Also AML with \proteinnamehuman{WT1} mutations exhibits genomic hypermethylation due to insufficient \proteinnamehuman{Tet-2} activity\cite{Rampal2014}.

Taken together, a differentiation block mediated by hypermethylation of the promoters of lineage regulating genes constitutes an important mechanism implicated in leukemogenesis of AML and thus deserves a closer look. Hypothetically, the \dnmtchip genotype could abate the faithful maintenance and inheritance of hypermethylation across cell divisions.  

Based on our WGBS results, we had dismissed hypermethylation completely, since both leukemia were hypomethylated in comparison to the previously published methylome of mouse hematopoietic stem cells (HSC)\cite{Jeong2014}\reffigurepage{fig:wgbs_violinplot}{} and also focal hypermethylations were virtually absent at base resolution. However, the strain-matched \mmsvjae \dnmtwt HSC methylome provided by our collaborator Daniel Lipka was hypomethylated by approximately \SI{10}{\percent}  compared to the \mmblsix HSC data\reffigurepage{fig:wgbs_sliding_windows1}{, left panel} vs.\reffigurepage{fig:wgbs_healthy_cchip3}{, top left panel}. Hence, the occurence of focal hypermethylations relative to the HSCs appeared more likely than before. 

Corroboration of hypermethylation was not possible, since the advance data sharing agreement with our collaborator Daniel Lipka did not comprise base-resolution access to the unpublished \mmsvjae \dnmtwt and \dnmtchip methylomes. Yet, corresponding tests are encouraged after availability. 