\chapter{Enhancers and other cis-regulatory elements}
\minitoc

\section{Gene-regulatory genomics}
\label{chap:i:enhancers:regulatorygenomics}

Protein-coding genes occupy only marginal fractions of the total sequence length in most eukaryotic genomes\cite{bookshelf-eukaryotic-genomes}: For those of mouse and human it is within \SIrange{2}{3}{\percent}. Scientists initially demeaned the remainder as ''junk DNA'' due to larger stretches of repetitive sequence and parts of viral origin bearing witness to the developmental history \cite{Makalowski2000,Palazzo2014}. From an evolutionary perspective, most species have favored genome integrity over active reduction of surplus sequence, unless nutrient poor conditions force to economize on phosphorous, nitrogen and carbon during DNA replication. The latter is exemplified by the genome of the carnivorous aquatic plant \emphspecies{Utricularia gibba}, whose \num{28000} genes account for \SI{97}{\percent} of its just \SI{82}{\mega bp} (mega-base pairs) genome size \cite{Ibarra-Laclette2013,Carretero-Paulet2015}. The human genome is \SI{40}{x} larger (\SI{3200}{\mega bp}), but comprises just moderately more genes (\num{46500}). A mouse genome is highly similar in size and gene content, but both are easily surpassed by plant and amphibian genomes, which may reach sizes of \SI{150}{\giga bp} or more \cite{Dufresne2011}.

This highly variable content of non-protein-coding DNA sparked a still ongoing lively debate over the amount of functional DNA in our genome\cite{Graur2013,Doolittle2013,Niu2013}. The better part of mouse and human genomes is to a variable degree transcribed into RNA \cite{Carninci2005,Djebali2012}, which however does not necessarily imply function\cite{Palazzo2015}. As it has been estimated that just \SI{8.2}{\percent} of the human genome is presently subject to negative selection\cite{Rands2014}, thus faithfully preserved throughout evolutionary history, the truly functional share might be in the single digit range. 

In any case the functional fraction comprises the protein-coding genes, various operational RNA molecules (rRNA, tRNA, snRNA, piRNA, lincRNA and others), rather structural parts like telomeres and centromeres and also non-coding parts of DNA that are ''regulatory''\cite{Ward2012}. In the broadest sense, the regulatory parts switch genes on and off or modulate the amount of transcript or protein output and are vital for the establishment of tissue- and/or cell-specific expression. Some regulation occurs after transcription took place, like that by microRNAs (miRNA), but a lot of control is directly exerted on the stage of messengerRNA (mRNA) production. Most of these mechanisms require the presence of regulatory sequences in the vicinity of the gene to be addressed. Such sequences are termed \emph{cis-regulatory elements}, because they are located on the same DNA molecule before, within or after the gene itself. To act on the gene, they in general make contact with its promoter resulting in a DNA loop\cite{Noordermeer2013,Grubert2015}. Typically one gene is targeted by several cis-regulatory elements and one element may also be involved in the regulation of different genes\cite{Bertolino2016,Javierre2016,Allahyar2018}.
 
\subsection{Classes of cis-regulatory elements}
\label{chap:i:enhancers:cisclasses}

The repertoire of cis-regulatory elements is quite diverse and three major categories, namely enhancer, silencer and insulator have been distinguished classically. The locus control regions are sometimes considered to be another separate category, however they altogether may be revised in future, since new findings challenge the classical schema: Next-generation sequencing based methods allowed to scrutinize thousands of regulatory elements in parallel and in many different cell types and developmental states. These investigations casted doubts on the concept that a particular element inherently functions in one determined way. Instead they supported the notion that cis-regulatory elements possess a regulative potential (which may vary between individual elements) and the epigenetic state, chromatin conformation and expressed transcription factors ultimately decide the outcome\cite{Stampfel2015}. 

\begin{mydescription}{0.5cm}
	\item[Enhancers] A regulatory sequence, which will orientation-independently increase the transcription rate of nearby genes when active\cite{Marsman2012,Gibcus2013}. It recruits positive transcription activators, proteins with the capability to support the production of the mRNA transcript \dissref{chap:i:enhancers:enhancerdef}.
	
	\item[Silencers] A sequence-specific element that exerts a negative effect on the transcriptional output of the targeted gene. By recruitment of repressive factors (like the KRAB zinc finger proteins\cite{Witzgall1994,Urrutia2003}) silencers interfere with the correct assembly of the transcription preinitiation complex (PIC) at the gene promoter\cite{Ogbourne1998}, a basal protein complex that is required to transcribe DNA sequence into RNA. Silencing can also be achieved by blockage of RNA elongation, as in the osteocalcin gene, where the relative abundance of two competitively binding proteins decides the outcome \cite{Frenkel1994,Li1995}. Particular CpG islands may form another group of negative regulatory sequences and serve as polycomb response elements (PREs), although the existence of PREs in mammals (contrary to \emphspecies{Drosophila}, where they are well characterized) remains debated\cite{Kassis2013,Bauer2016,Rickels2017}. Silencers, like enhancers, act in an orientation-, position-, and distance-independent manner.
	
	\item[Insulators] DNA sequence elements that shield a gene's promoter from the ramifications of nearby cis-regulatory elements or serve as heterochromatin boundary. Adjacent repressive chromatin tends to propagate into a transcriptionally active euchromatic region unless an insulator element protects it\cite{Donze2002}. The active chromatin configuration is maintained by recruitment of various insulating proteins like \proteinnamehuman{USF1}, \proteinnamehuman{VEZF1} and potentially \proteinnamehuman{CTCF}\cite{Gaszner2006,Barkess2012,Ong2014}. Furthermore insulators can exhibit a cis-element-blocking function, when they are located in between a promoter and an enhancer or silencer. The chicken \ensuremath{\beta}-globin insulator (cHS4) is a well studied vertebrate insulator, which possesses both heterochromatic barrier, silencer- and enhancer-blocking capabilities\cite{Walters1999,Yao2003,Rincon-Arano2007}. The advent of next-generation sequencing based methods to quantify incidences of chromosomal contacts (particularly HiC\cite{Lieberman-Aiden2009}) has introduced the concept of topologically associating domains (TADs)\cite{Dixon2012,Jost2014}\citerev{Sexton2015}. Chromosomal contacts are frequent within a TAD, but rarely occur outside of it. Nevertheless not all TAD boundaries qualify as classical insulators, as they can for example also occur at housekeeping gene promoters\cite{Hug2017,Flyamer2017}. Furthermore HiC analyses after \proteinnamehuman{CTCF} knockdown have shown that \proteinnamehuman{CTCF} mediates transcriptional insulator function through enhancer blocking, is implicated in TAD organization and chromatin folding but not relevant to rein heterochromatin spreading \cite{Nora2017}. Thus, shielding promoters from nearby cis-regulatory elements and limiting heterochromatin spreading might be independent functions. 
	
	\item [Locus control region] It opens condensed heterochromatic domains and primes whole gene clusters for activation. Once a LCR has induced conformational change of chromatin, local enhancers within the cluster sustain and fine-tune the transcription of the genes.
	Locus control regions act over large distances as long-term on-switches in the chromosome, since the reestablishment of the repressed chromatin state requires the cell to pass through S-phase\cite{Li2002}. In experiments, LCRs, which have been artificially cloned outside of their native context, subsume properties of both transcriptional enhancers and chromatin insulators. It is however disputed, whether they exert all of those functions in a regular chromosomal environment and if they should merely count as strong enhancers\cite{Bulger2013}. A well-studied LCR activates the \ensuremath{\beta}-globin locus when erythroid lineage commitment takes place\cite{Forrester1990,Jimenez1992,Allahyar2018}. Newer publications coined the terms \emph{super enhancer}\cite{Hnisz2013} or \emph{stretch enhancer}, which were also applied on known LCR regions. Thus, both names are mostly synonymous, although LCRs may be a more specific subset. 
	
	\item[Other] On top of that there are various further motifs and genomic segments , which can justifiably considered to be potential separate entities of cis-regulatory or cis-acting elements. This includes response elements to a variety of factors like hormones, cytokines (interferones, interleukins), hypoxia-inducible factors, retinoic acid or vitamin D. Furthermore upstream open reading frames (uORFs) and upstream AUG (uAUGs) located in the 5\lq UTR of mRNAs \cite{Iacono2005,Kochetov2008} as well as elements for splicing control\cite{Wang2006} shall be mentioned briefly. 
\end{mydescription} 



\section{Characteristics of eukaryotic enhancers}
\label{chap:i:enhancers:enhancerdef}

Operationally, enhancers augment the activity of a nearby promoter and are enriched in recognition motifs for sequence-specific transcription factors. Since the orientation and to a large extent also the exact position relative to the promoter is insignificant, they are considered to be orientation- and position-independent. Anyhow, most enhancers are thought to reside in the vicinity of the targeted promoter, either a few thousand bases upstream or (often in the first two introns of the gene) downstream. However, also long-range promoter contacts are common\cite{Sanyal2012,Mifsud2015}, as illustrated by the regulatory framework of the \proteinnamehuman{myc} locus \cite{Fulco2016} or the \proteinnamehuman{shh} gene\cite{Lettice2003}. 

\subsection{Mode of action} 
\label{chap:i:enhancers:moa}
The mechanism of enhancer action requires a change of the three-dimensional chromatin structure and the formation of a DNA loop. In doing so, enhancers increase the concentration of transcriptional activators near the promoter, but the temporal progression of the process is poorly understood. The discovery of preexisting chromatin looping, which precedes the actual signaling\cite{Jin2013} has challenged the previous model\cite{Arensbergen2014} that solely specific transcription factors govern the looping\cite{Drissen2004,Vakoc2005}. Furthermore formation and dissociation of loops must undergo tight sequential coordination as one gene is typically targeted by several cis-regulatory elements and one element may also be involved in the regulation of different genes\cite{Hughes2014,Bertolino2016,Javierre2016}. Embryonic development\cite{Ho2015} and hematopoietic lineage commitment\cite{Kieffer-Kwon2013} exemplify the latter. 

Once enhancer and promoter have converged, coactivators facilitate protein–protein interactions\cite{Kaiser1996}, modulate the activity of transcription factors, alter epigenetic marks on histones (which aid the decompaction of the chromatin fiber)\cite{Ogryzko1996,Spencer1997} or recruit kinases to phosphorylate the c-terminal tail of \proteinname{RNA polymerase II}\cite{Hirose2007}. Out of the hundreds of different coactivators in the mouse and human genome, more than fifty operate on any given transcript start site (TSS) in conjunction. The centerpiece of the coactivator structure is the mediator complex\cite{Mittler2001}, which is conserved from yeast to human\cite{Kornberg2005,Robinson2016}. For its identification and characterization, the Nobel Prize in Chemistry was awarded to Roger D. Kornberg in 2006.

Bringing enhancer-bound protein factors close to the promoter-bound preinitiation complex (PIC) is the best characterized function of enhancers, but not the sole. Besides proteins, also regulatory RNAs take part in the molecular machinery of eukaryotic transcription, some of which are linked to enhancers. Pol II-transcribed enhancer RNAs (eRNAs) are the most prominent example \dissref{chap:i:enhancers:erna}, but also the Pol III-transcribed non-coding 7SK small nuclear RNA (7SK) can be recruited by enhancers\cite{CQuaresma2016}. 7SK has a scaffold function for an inhibitory 7SK small nuclear ribonucleoprotein (7SK snRNP) complex, which in its canonical form suppresses \proteinnamemouse{P-TEFb}-mediated release of \proteinname{RNA polymerase II pausing} at promoters\cite{Core2008,Jonkers2014}. At enhancers an alternate assembly of 7SK-complex dominates, which incorporates members of the BAF chromatin-remodeling complex and inhibits enhancer RNA transcription by modulating nucleosome position\cite{Flynn2016}. As the two 7SK complexes are mutually exclusive\cite{CQuaresma2016}, it is tempting to speculate that the enhancer-bound 7SK–BAF snRNP repels the canonical 7SK snRNP and permits release of promoter-proximal pausing. Indeed, the existence of anti-pause enhancers has been shown\cite{Liu2013}, however the predominant mechanisms seem to be \proteinnamehuman{Brd4} and \proteinnamehuman{JMJD6} recruitment for destabilization of 7SK by decapping\cite{Ong2011,Liu2013} and acting as decoy for \proteinnamehuman{NELF}\cite{Schaukowitch2014}.

It should be emphasized that the enhancer characteristics outlined in this section are largely limited to the genomes of higher eukaryotes. While sea urchins, which developed about 500 million years ago, employ enhancers\cite{Grosschedl1980,Palla1994} and insulators\cite{Palla1997} for transcriptional regulation, lower eukaryotes such as yeast rely on upstream activator sequences (UAS)\cite{West1984,Bram1986,Webster1988}. They are somewhat analogous to enhancers of higher eukaryotes, but differ in their inability to activate a promoter from downstream positions. They do however function in either orientation and at variable distances from the promoter. In bacteria, a distinct form of RNA polymerase containing sigma factor 54 (\ensuremath{\sigma^{54}}) recognizes simple enhancers upstream of a promoter\cite{Morett1993}. Viral genomes like those of the simian virus 40 (SV40) may contain own enhancers, which will become functional after integration into the host genome\cite{Khoury1983,Zenke1986,Ondek1988}. 

% % % % Evolution of ALU elements \cite{Su2014} 

\subsection{Enhancer states and activation} 
\label{chap:i:enhancers:states}

Before next-generation sequencing based methods were developed to identify potential enhancers genome-wide\dissref{chap:i:enhancers:methods}, artificial expression systems were used to validate the ability of a DNA sequence to enhance transcription. The elements to be tested were cloned adjacent to a weak promoter, which typically originated from a different gene than the one initially associated with the regulatory region\cite{Banerji1983,Gillies1983}. When the transcription rate was quantified as readout in vitro or in vivo, even such heterologous transcripts would often recapitulate known tissue-specific regulation. Thus, selective activation was mostly imposed by the enhancer\cite{Grosveld1987,Greaves1989}. 

The core promoter seemed to define temporal windows with the opportunity for activation, subsequently precise expression timing was imposed by enhancers \cite{Batut2017}. This finding raised the question how enhancers are regulated. Today it is widely accepted that the correct subset is selected from the repertoire of potential enhancers by means of higher-order chromatin organization and specific pioneering transcription factors, which open up condensed chromatin and expose the enhancer to the binding of additional factors. Once operable, the formation of chromatin loops, changes in methylation and the recruitment of further transcriptional activators constitute further regulatory layers. Four enhancer states can be distinguished according to Heinz and colleagues\cite{Heinz2015}:

\begin{mydescription}{0.5cm}
	\item[Closed] The cis-regulatory sequence is situated in highly compacted chromatin and is inaccessible for transcription factor binding. Furthermore it lacks characteristic histone modifications other than those associated with heterochromatin. Enhancers, which are closed in stem cells and only appear in differentiated cells are referred to as latent enhancers.
	\item[Primed] Pioneering transcription factors have facilitated the loosening of the compacted chromatin\cite{Mayran2018} and a nucleosome-free region of accessible open DNA was established. This enables the binding of cooperating sequence-specific transcription factors and the recruitment of coactivators. Nevertheless essential cues for activation are missing. 
	\item[Poised] This state is in many regards comparable to a primed enhancer, but nearby repressive epigenetic chromatin marks like \histwentyseventhree quiesce the site additionally. Since they enable rapid activation of a particular developmental program, poised enhancers are often found in embryonic stem cells and to a lesser extent also in tissue-specific stem cells. 
	\item[Active] The core of an activated enhancer is nucleosome-free, open chromatin permits the binding of transcription factors and coactivators. Flanking nucleosomes are typically marked by \histwentysevenac\cite{Creyghton2010} and eRNAs are produced at active enhancers. 
\end{mydescription}

The signature of histone marks\cite{Ernst2010,CaloWysocka2013} and bound coactivators as well as the enhancer's propensity to generate eRNA transcripts is highly relevant. Since some of the next-generation sequencing based methods \dissref{chap:i:enhancers:methods} rely on these patterns to identify enhancers genome-wide, sensitivity and specificity of the respective method will vary and sometimes confine itself to enhancers in a particular state. 


\subsection{Enhancer RNAs}
\label{chap:i:enhancers:erna}

In retrospect, Northern Blot experiments of the \ensuremath{\beta}-globin locus control region showed transcription of enhancer elements in 1990 already, however the results were mistaken for run-through main transcript and attributed defective polyadenylation\cite{Collis1990}. Two decades later, a study in neuronal cells uncovered bidirectional transcripts initiated at known enhancers with occupancy of co-activators and termed them enhancer RNA\cite{Kim2010}. Around the same time another group confirmed these observations in macrophages\cite{DeSanta2010}. The production of eRNAs was evidently restricted to active enhancers, since the transcript start sites were reported to be enriched in \hisfourone and \histwentysevenac, but devoid of \histwentyseventhree. If those are unavailable, \hiseightac, \hisnineac and the elongation associated \histhirtysixthree \& \hisseventyninetwo can serve as surrogate markers for eRNA production\cite{Zhu2013a}.

Enhancer RNAs were soon shown to be capped on the 5\lq end, short (\textless\SI{1}{kb}), bidirectional, unspliced and rapidly degraded by the exosome\cite{Koch2011,Andersson2014,Core2014}, which contested a relevant functional role of the pervasive transcription initiating from enhancers. Contrarily, eRNAs have been demonstrated to stabilize enhancer–promoter association and chromatin loops at steroid hormone response genes\cite{Wang2011,Li2013a}, to be of importance for \hisfourone and \hisfourtwo deposition by \proteinnamehuman{Mll3} and \proteinnamehuman{Mll4} at \emph{de novo} enhancers\cite{Kaikkonen2013} and to be subject to functional methylation\cite{Aguilo2016}. Thus, they exert meaningful roles at least for a subset of enhancers \citerev{Lam2014} and their expression generally correlates with the that of their target genes\cite{Arner2015}.

Especially the relationship and interplay of enhancer RNAs (eRNAs) and long-noncoding RNAs (lncRNAs) challenged the distinctness of enhancers and promoters and has been subject to an ongoing scientific dispute\cite{Kowalczyk2012}. While the ability of certain lncRNAs to enhance the expression of nearby genes\cite{Oerom2010} by preventing DNA methylation\cite{DiRuscio2013} and maintaining active chromatin states\cite{Yang2014} was known, the finding that the majority of them originated from enhancer-like elements came by surprise [\citenum{Su2014a}, reviewed in \citenum{Chen2017d}]. Furthermore \proteinnamemouse{Kdm5c} was shown to regulate whether individual elements rather exhibited enhancer-like or promoter-like activities\cite{Outchkourov2013}. Previously the lack of splice donors\cite{Fong2001} proximal to enhancers was believed to preclude productive elongation of eRNAs\cite{Core2014}. Instead, it was shown that the adapter protein \proteinnamehuman{WDR82} and its associated complexes actively terminate the transcription. After knock-down of complex proteins, many eRNAs were actively elongated into lncRNAs\cite{Austenaa2015}. This finding also provides a rationale for the presence of the 7SK–BAF snRNP\dissref{chap:i:enhancers:moa} at enhancers. Convergent transcription, which would inevitably occur upon elongation of eRNAs at clustered elements within super enhancers, needs to be suppressed as it would otherwise trigger strong DNA-damage signaling\cite{Meng2014,Flynn2016}.

In summary, broad similarities between enhancers and promoters exist: Both may initiate directed or undirected transcription. Interacting regulatory complexes as well as the surrounding sequences determine the length and stability of the transcript\cite{Andersson2015,Andersson2015a,Haberle2018}. 

\section{Methods for genome-wide identification of enhancers}
\label{chap:i:enhancers:methods}

Next-generation sequencing techniques can be used to predict and annotate enhancers in various ways. An analysis may comprise the prediction of enhancers, the determination of cell-type-specific activity and the linking to their target genes\cite{Lim2018,Ramisch2019}. A variable degree of inaccuracy and computational pitfalls inhere in each method, which are outlined in the description for the respective technique. Multiple genome-wide assays can be combined to boost confidence\cite{Benton2019}, but as the sensitivity of the methods is highly variable, a lot of potential enhancers are wrongfully eliminated when the consensus is used only. For example many enhancers require their native chromatin context to function properly\cite{Inoue2016} and will not show activity in reporter assays as exemplified by the upstream regulatory element (\proteinnamehuman{URE}) of \proteinnamehuman{PU.1}\cite{Li2001,Rosenbauer2004,Rosenbauer2006,Leddin2011}. On the other hand even experimental deletion or mutagenesis of a bona-fide enhancer might not result in gene expression changes due to frequent redundancy of cis-regulatory elements\cite{Osterwalder2018}. These challenges might explain the large range of estimates on the total number of enhancers in the genome to be found in the literature.

The most commonly used methods for enhancer detection are listed below\citerev{Shlyueva2014,Whitaker2015}:

\begin{mydescription}{0.5cm}
	\item[Open chromatin] To function and permit binding of transcription factors or coactivators all classes of cis-regulatory elements need to be nucleosome-free in their active state. Open chromatin however is also readily accessible for enzymatic digestion by nucelases or integration by transposases. Protocols to map nuclease hypersensitive sites\cite{Stewart1991} have underpinned the discovery of cis-regulatory elements for decades and were among the first techniques, which were adapted to next-generation sequencing\cite{Crawford2006}. The assay was widely used during the ENCODE project\cite{Thurman2012}, but like the related FAIRE-seq (Formaldehyde Assisted Isolation of Regulatory Elements)\cite{Giresi2007} it is nowadays mostly obsolete. Both have been replaced by ATAC-seq (Assay for Transposase Accessible Chromatin)\cite{Buenrostro2013}, which features simpler library preparation protocols and can downscale to single cells\cite{Buenrostro2015,Cusanovich2015}. All three methods however do not determine the class of the cis-regulatory element in question and require a comprehensive annotation of genes (promoters,exons) for exclusion. 
	\item[Histone modifications] The nucleosomes flanking an enhancer sequence typically undergo epigenetic alterations, which constitute a relatively distinctive pattern. If ChIP-seq experiments (Chromatin ImmunoPrecipitation)\cite{Barski2007,Johnson2007,Mikkelsen2007} for all those modifications are carried out, putative enhancers can be identified in several different states\dissref{chap:i:enhancers:states} at once. The discrimination between active and poised enhancers based on \histwentysevenac and \histwentyseventhree respectively\cite{Zentner2011} is well established, but the guidelines for methylation on lysine 4 are conflicting. A widely cited publication suggested to define enhancers based on presence of \hisfourone and lack of \hisfourthree\cite{Heintzman2007}. Although both modifications attach to the same lysine residue and are mutually exclusive, each nucleosome consists of two H3 proteins and several nearby nucleosomes can be modified accordingly. Thus, both modifications could technically be detectable close to the same enhancer in parallel. Indeed concomitant \hisfourone, \hisfourtwo and \hisfourthree modifications have been reported at enhancers\cite{Pekowska2011,KochAndrau2011} and recent experiments in \emphspecies{Drosophila} embryos and mouse embryonic stem cells (mESCs) indicate that all methylation states are functionally interconvertible\cite{Rickels2017a}. Hence an ambiguous search pattern is a disadvantage of the method, furthermore the size of the nucleosome limits the spatial resolution to about \SI{2}{kb} (with exceptions \cite{Rhee2011}), which impedes motif analyses of the core enhancer. On the other hand high quality antibodies to pull down modified histone residues, standardized protocols and kits are commercially available. Thus, such data can be generated with a fair level of expertise from few cells \cite{Lara-Astiaso2014} down to single cell level\cite{Rotem2015}. 
	\item[Coactivator ChIP] If the lineage-determining transcription factors governing the differentiation into a specific cell type are known, their binding to gene-distant sites can guide the characterization of cis-regulatory elements and vice versa\cite{Bornstein2014,Bertolino2016}. ChIP-seq of cofactors such as the histone acetyltransferase p300\cite{Visel2009} or specific subunits of the mediator complex\cite{Aranda-Orgilles2016} can also lead to the discovery of new enhancers. Since immunoprecipitation of transcription factors is generally challenging and specific monoclonal antibodies are often unavailable, this approach is rarely chosen to identify enhancers. Furthermore, it has been suggested that transcription factor occupancy does not equal function\cite{Lickwar2012} and false-positive signals are quite common\cite{Teytelman2013}. 
	\item[eRNA profiling] The transcription of eRNAs\dissref{chap:i:enhancers:erna} is indicative of active enhancer function. GRO-seq, which assesses nascent transcription\cite{Gatehouse1995,Core2008}, as well as CAGE-seq\cite{Carninci1996,Shiraki2003}, which maps the 5\lq transcription start sites of mRNAs, can be used to detect enhancers. However, both computational analyses are intricate and potentially prone to a large fraction of false-positives\cite{Andersson2014,Nagari2017}. A huge advantage is the promoter-specific simultaneous gene expression profiling\cite{Takahashi2012}, which simplifies the identification of enhancer target genes and somewhat compensates for the challenging library preparation. Furthermore the exact location of the core enhancer can be traced with unparalleled precision due to the narrow peaks. 
	\item[Sequence-based] To a certain extent, sites of enhancers can be derived purely from the underlying sequence, if transcription factor binding motifs are known and reference genomes of some reasonably related species are additionally available for comparison. Although cross-species conservation of individual enhancers is low\cite{Schmidt2010,Blow2010,Villar2015}, frequent motif patterns and higher-than-average sequence constraints are suggestive of cis-regulatory elements\citerev{Long2016}. Computational tools, such as the \emphsoftwarename{Enhancer Element Locator}\cite{Hallikas2006} can be used to screen for genes regulated by certain transcription factors. The opposite analysis is performed by \emphsoftwarename{kmer-SVM}\cite{Fletez-Brant2013} or \emphsoftwarename{HOMER}\cite{Heinz2010}, which will detect recurring motifs in a given set of cis-regulatory elements. 
	\item[Reporter-based] As mentioned previously, some enhancers require their physiological native chromatin environment to function properly \cite{Inoue2016}. For others this premise is negligible and they can be successfully cloned into reporter constructs for screening. Typically the enhancers sustain the expression of a reporter gene, such as a fluorescent protein or an enzyme to power a photometric or colorimetric reaction\citerev{Inoue2015}. Next-generation sequencing has enabled high-throughput variants of those assays, one of which is STARR-seq\cite{Arnold2013}. It was gradually adapted to mammalian genomes\cite{Vanhille2015,Liu2017} and features a transcript containing the sequence under investigation. Thus, the enhancer element is contained in the transcript and can be recovered from a RNA sequencing. A drawback of many reporter-based assays is the transcription initiated at the bacterial ORI\cite{Lemp2012}, which is present in many vectors, as well as the triggering of an interferon response in the transfected target cells\cite{Muerdter2018}.
\end{mydescription}

The Rosenbauer lab is proficient in the use of four different methods of various categories to detect enhancers: \hisfourone/\histwentysevenac ChIP-seq, CAGE-seq eRNA detection, STARR-seq and ATAC-seq. 

\section{Enhancer involvement in pathogenesis}
\label{chap:i:enhancers:pathogenesis}

The majority of known sequence variants that contribute to disease\footnote{according to the \emphdatabasename{NCBI ClinVar} database, normalized to sequence length} can be found in \emphdatabasename{ENCODE}-annotated cis-regulatory elements. Mutations within those sites may abrogate their function or severely alter their properties, but predicting the consequences in non-coding DNA is challenging\cite{Kwasnieski2012,Tewhey2016}. In contrast, evaluation of a specific nucleotide change in protein-coding DNA is relatively straightforward on the grounds of the known codon table.

% http://genetics.wustl.edu/bclab/research/ for example
Several laboratories have devoted themselves to establish an analogous \emph{cis-regulatory grammar}\cite{White2015}, which is believed to exist, because sequence features in enhancers and the arrangements of transcription factor binding sites often conform to certain physical constraints\cite{Zeigler2014}. The non-coding DNA grammar is clearly neither as universal nor as distinct as the codon code, but various techniques such as \emphsoftwarename{Thermodynamic State Ensemble Models}\cite{Sherman2012} have already contributed to formalize some of the rules. A better understanding of such instructions does not only aid in interpreting disease causing genetic variation in non-coding DNA, but also helps to understand the exaptation of transposable elements into novel cis-regulatory sites\cite{Souza2013,Long2016}. 

It should be stressed that also aberrant epigenetic modifications may disrupt proper regulation and thus function additionally or jointly with mutations in a pathogenic manner\cite{Planello2016}.

\subsection{Examples of pathogenic enhancer abnormalities}
\label{chap:i:enhancers:pathogenicabnorm}

Substantial knowledge on the basic principles of genetic regulation was derived from studies at the \ensuremath{\beta}-globin gene cluster and so were the first indications for pathogenic cis-regulatory effects. DNA from thalassaemia patients was tested for genetic rearrangements, but sometimes the integrity of the \ensuremath{\beta}-globin gene itself was untampered with. In one case study the gene was aberrantly hypermethylated and silenced in vivo due to an unspecified cis-effect\cite{Kioussis1983}. Another case study a few years later demonstrated that a particular \SI{100}{kb} deletion contained two Alu elements\cite{Taramelli1986}, whose enhancer activity could be shown in vitro. Furthermore the abrogation of the locus control region will also lead to thalassaemia\cite{Harteveld2005}. In all named cases the pathogenic phenotype was a result of insufficient \ensuremath{\beta}-globin protein levels. 

In this sense, the clearest causalities arise from the disturbance of genes that are closely linked to a certain disease. Pre-axial polydactyly, a Mendelian disorder caused by ectopic \proteinnamehuman{SHH} expression\cite{Lettice2003} is another example of a straightforward causality. However, in accordance with cell-type or tissue-specific enhancer activity, an enhancer-mediated pathogenesis may not fully recapitulate the clinical phenotype of patients who carry mutations in the genes itself. This is illustrated by the Holt-Oram syndrome: Pathogenic \proteinnamehuman{TBX5} mutations will typically present itself by limb malformations and heart defects, whereas a disrupted enhancer may only affect the cardiac expression and permit normal limb development\cite{Smemo2012}.

The contribution of enhancers to multifactorial, non-Mendelian medical conditions is even harder to assess, but might be nevertheless unraveled by association studies. Such a study proved that the susceptibility for Hirschsprung’s disease strongly increased by an incapacitated intronic enhancer of the \proteinnamehuman{RET} tyrosine kinase, despite low penetrance and different genetic effects in males and females \cite{Emison2005}. 

Association studies were formerly conducted mostly between relatives from families with an elevated risk for a certain condition and assayed only few genetic loci. The advent of microarrays and subsequently the era of next-generation sequencing has shifted the attention to genome-wide association studies (GWAS), which screen large patient vs. control cohorts and test thousands of genetic variants in parallel - e.g. for Alzheimer's disease\cite{Kikuchi2019}.

This unbiased approach allows to detect alterations also in unsuspected distant cis-regulatory regions, but is prone to false-positives due to random associations within the large number of tested regions. Thus, considerable controversy sparked over the benefits and accuracy of GWAS, especially in terms of risk assessment and counseling in personalized genetic testing\cite{Heshka2008}. 

Strategies to undergird low-penetrance risk variants include experimental validation by CRISPR-mediated targeted mutation\cite{Fulco2016} or the quest for subgroup-specificity\cite{Lin2016} and recurring association with different diseases. The latter is exemplified by the single-nucleotide polymorphism rs6983267, which lies within a transcriptional enhancer of the \genenamehuman{MYC} proto-oncogene and increases the risk for colorectal as well as prostate cancer\cite{Pomerantz2009,Wasserman2010} due to a higher responsiveness to Wnt-signaling\cite{Tuupanen2009}. Nevertheless the determination of enhancer effects on susceptibility to cancer remains challenging\citerev{Herz2014,Smith2014,Sur2016}.

\subsection{Enhancers contribute to leukemogenesis}
\label{chap:i:enhancers:leukemia}

Hematopoiesis, the development of diverse mature blood cells from hematopoietic stem cells requires an intricate regulation. The appropriate expression of key transcription factors such as \proteinnamehuman{PU.1}, \proteinnamehuman{GATA1}, \proteinnamehuman{GATA2} or \tfcebpa at various stages governs progenitor commitment and differentiation. Ten-thousands of enhancers are presumably involved in hematopoietic regulation in total\cite{Lara-Astiaso2014,Ulirsch2019,Bresnick2019}.

Generalizations about leukemogenesis are almost futile, given the many different subtypes . However, in some cases notorious genes like \genenamehuman{MYC} and its enhancers, which are repeatedly implicated in cancerogenesis, do affect other cancers as well as leukemia\cite{Zuber2011,Shi2013}. In contrast, the downregulation of \proteinnamehuman{PU.1} is restricted to hematopoietic cancerogenesis. None the less, it represents a proven route to leukemia\cite{Rosenbauer2004,Metcalf2006} and already subtle \proteinnamehuman{PU.1} reduction by a heterozygous deletion of an enhancer was sufficient to initiate a myeloid-biased preleukemic state\cite{Will2015}.

Especially late-onset leukemia are characterized by the presence of preleukemic hematopoietic stem cells, which have progressively acquired an increasing mutation burden over their lifetime. These cells are not yet leukemic and expansive, but exhibit spurious alterations in their gene expression programs and enhancers. Such genetic regulation patterns mimicking disparate developmental stages, ultimately increase susceptance to uncontrolled cellular expansion and are retained in the descendant leukemic clones, which was revealed by a study in late acute myeloid leukemia (AML)\cite{Corces2016}. 

Unsurprisingly, aberrant super enhancers (respectively LCRs) strongly promote of preleukemic states, since they govern the activation of whole gene clusters. This has been elaborated with regard to T-cells and the pathogenesis of T cell acute lymphoblastic leukemia (T-ALL): The introduction of binding motifs for the \proteinnamehuman{MYB} transcription factor by somatic mutations forms a novel super enhancer upstream of the TAL1 oncogene and sustains its expression\cite{Mansour2014,Vahedi2015}. In a particularly dismal ALL subtype driven by \proteinnamehuman{TCF3-HLF}, the chimeric transcription factor activates an enhancer cluster controlling expression of the \genenamehuman{MYC} gene and instigates the respective transcriptional program\cite{Huang2019}. Because hematopoietic \genenamehuman{MYC} expression is intricately regulated by combinatorial and additive activity of individual enhancer modules within this cluster\cite{Bahr2018}, a dysregulation of \genenamehuman{MYC} program can be mediated by various factors or arise from amplifications within the enhancer region\cite{Shi2013}. Therefore, the enhancer cluster is complicated in many leukemia subtypes and also pivotal for \mllafnine-driven AML\cite{Bahr2018}. 

A different mode of action has been reported for a distinct subtype of acute myeloid leukemia. In AML with the \emph{inv(3)(q21;q26)} karyotype\cite{Arber2016} a genomic rearrangement repositions a distal hematopoietic enhancer of \proteinnamehuman{GATA2} in close proximity to the stem-cell regulator \proteinnamehuman{EVI1}, which is ectopically activated. Concomitantly, \proteinnamehuman{GATA2} expression is diminished and both events facilitate leukemic expansion\cite{Yamazaki2014,Groeschel2014}. 




